<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Katherine Eaton" />
  <meta name="author" content="Leo Featherstone" />
  <meta name="author" content="Sebastian Duchene" />
  <meta name="author" content="Hendrik Poinar" />
  <meta name="dcterms.date" content="2021-06-17" />
  <meta name="keywords" content="markdown, publishing, manubot" />
  <title>Plagued by a cryptic clock</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="Plagued by a cryptic clock" />
  <meta name="citation_title" content="Plagued by a cryptic clock" />
  <meta property="og:title" content="Plagued by a cryptic clock" />
  <meta property="twitter:title" content="Plagued by a cryptic clock" />
  <meta name="dc.date" content="2021-06-17" />
  <meta name="citation_publication_date" content="2021-06-17" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Katherine Eaton" />
  <meta name="citation_author_institution" content="Ancient DNA Centre, McMaster University" />
  <meta name="citation_author_institution" content="Department of Anthropology, McMaster University" />
  <meta name="citation_author_orcid" content="0000-0001-6862-7756" />
  <meta name="citation_author" content="Leo Featherstone" />
  <meta name="citation_author_institution" content="The Peter Doherty Institute For Infection and Immunity, University of Melbourne" />
  <meta name="citation_author_orcid" content="0000-0002-8878-1758" />
  <meta name="citation_author" content="Sebastian Duchene" />
  <meta name="citation_author_institution" content="The Peter Doherty Institute For Infection and Immunity, University of Melbourne" />
  <meta name="citation_author_orcid" content="0000-0002-2863-0907" />
  <meta name="citation_author" content="Hendrik Poinar" />
  <meta name="citation_author_institution" content="Ancient DNA Centre, McMaster University" />
  <meta name="citation_author_institution" content="Department of Anthropology, McMaster University" />
  <meta name="citation_author_orcid" content="0000-0002-0314-4160" />
  <link rel="canonical" href="https://ktmeaton.github.io/plague-global-phylo-paper/" />
  <meta property="og:url" content="https://ktmeaton.github.io/plague-global-phylo-paper/" />
  <meta property="twitter:url" content="https://ktmeaton.github.io/plague-global-phylo-paper/" />
  <meta name="citation_fulltext_html_url" content="https://ktmeaton.github.io/plague-global-phylo-paper/" />
  <meta name="citation_pdf_url" content="https://ktmeaton.github.io/plague-global-phylo-paper/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://ktmeaton.github.io/plague-global-phylo-paper/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://ktmeaton.github.io/plague-global-phylo-paper/v/fcf343b35e567fa8abdb55d7810bb3ba51e3f915/" />
  <meta name="manubot_html_url_versioned" content="https://ktmeaton.github.io/plague-global-phylo-paper/v/fcf343b35e567fa8abdb55d7810bb3ba51e3f915/" />
  <meta name="manubot_pdf_url_versioned" content="https://ktmeaton.github.io/plague-global-phylo-paper/v/fcf343b35e567fa8abdb55d7810bb3ba51e3f915/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Plagued by a cryptic clock</h1>
</header>
<p><subtitle><strong>New insights from the global phylogeny of <i>Yersinia pestis</i></strong></subtitle></p>
<p><small><em>
This manuscript
(<a href="https://ktmeaton.github.io/plague-global-phylo-paper/v/fcf343b35e567fa8abdb55d7810bb3ba51e3f915/">permalink</a>)
was automatically generated
from <a href="https://github.com/ktmeaton/plague-global-phylo-paper/tree/fcf343b35e567fa8abdb55d7810bb3ba51e3f915">ktmeaton/plague-global-phylo-paper@fcf343b</a>
on June 17, 2021.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Katherine Eaton</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6862-7756">0000-0001-6862-7756</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/ktmeaton">ktmeaton</a><br>
<small>
Ancient DNA Centre, McMaster University; Department of Anthropology, McMaster University
· Funded by Social Sciences and Humanities Research Council XXXXXXXX
</small></p></li>
<li><p><strong>Leo Featherstone</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-8878-1758">0000-0002-8878-1758</a><br>
<small>
The Peter Doherty Institute For Infection and Immunity, University of Melbourne
</small></p></li>
<li><p><strong>Sebastian Duchene</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-2863-0907">0000-0002-2863-0907</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/sebastianduchene">sebastianduchene</a><br>
<small>
The Peter Doherty Institute For Infection and Immunity, University of Melbourne
</small></p></li>
<li><p><strong>Hendrik Poinar</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0314-4160">0000-0002-0314-4160</a><br>
<small>
Ancient DNA Centre, McMaster University; Department of Anthropology, McMaster University
</small></p></li>
</ul>
<h2 class="page_break_before" id="abstract">Abstract</h2>
<h2 class="page_break_before" id="introduction">Introduction</h2>
<p>Plague has an impressively long and expansive history as a human disease. The earliest evidence of the plague bacterium, <em>Yersinia pestis</em>, comes from ancient DNA studies, dating its emergence to at least the Neolithic <span class="citation" data-cites="Vo6ReJPm AQa9Tn4j">[<a href="#ref-Vo6ReJPm" role="doc-biblioref">1</a>,<a href="#ref-AQa9Tn4j" role="doc-biblioref">2</a>]</span>. Since then, <em>Y. pestis</em> has traveled extensively due to ever-expanding global trade networks and the ability to infect a wide variety of mammalian hosts <span class="citation" data-cites="1093vihdz uM6Rh5Fu">[<a href="#ref-1093vihdz" role="doc-biblioref">3</a>,<a href="#ref-uM6Rh5Fu" role="doc-biblioref">4</a>]</span>. Few regions of the ancient and modern world remain untouched by this disease, as plague has an established presence on every continent except Oceania <span class="citation" data-cites="h4WAqbKy">[<a href="#ref-h4WAqbKy" role="doc-biblioref">5</a>]</span>.</p>
<p>Accompanying this prolific global presence is unnervingly high mortality. The infamous medieval Black Death is estimated to have killed more than half of Europe’s population <span class="citation" data-cites="13ET92iS0">[<a href="#ref-13ET92iS0" role="doc-biblioref">6</a>]</span>. This virulence can still be observed in the post-antibiotic era, where case fatality rates range from 22-71% <span class="citation" data-cites="Nd9WUU1j">[<a href="#ref-Nd9WUU1j" role="doc-biblioref">7</a>]</span>. As a result, plague maintains its status as a disease that is of vital importance to current public health initiatives.</p>
<p>This high priority disease status is unsurprising given that <em>Y. pestis</em> is a member of the Enterobacteriaceae family. This family includes other notable pathogens such as <em>Escherichia coli</em> and <em>Salmonella typhi</em> that are commonly transmitted by contaminated food and water. In comparison, the plague bacterium is unique among this family due to a striking difference in host habitat and transmission. <em>Y. pestis</em> commonly resides in the blood of its mammalian hosts and can be transmitted to new hosts through an infectious fleabite <span class="citation" data-cites="kQFNGkNf">[<a href="#ref-kQFNGkNf" role="doc-biblioref">8</a>]</span>. Furthermore, this unique mechanism evolved relatively recently, possibly around the 1st millennium BC <span class="citation" data-cites="ZvyYZ3sx">[<a href="#ref-ZvyYZ3sx" role="doc-biblioref">9</a>]</span>, long after <em>Y. pestis</em> emerged as a monomorphic clone of the enteric pathogen <em>Yersinia pseudotuberculosis</em> <span class="citation" data-cites="TV8iDhk7">[<a href="#ref-TV8iDhk7" role="doc-biblioref">10</a>]</span>.</p>
<p>While the population structure of <em>Y. pseudotuberculosis</em> is well-defined, <span class="citation" data-cites="RrDa6yZx 17ygZQMtK">[<a href="#ref-RrDa6yZx" role="doc-biblioref">11</a>,<a href="#ref-17ygZQMtK" role="doc-biblioref">12</a>]</span>, the phylogenetic patterning of <em>Y. pestis</em> remains cryptic. Populations of <em>Y. pestis</em> have been historically categorized according to a a vast array of historical, ecological, biochemical, and molecular characteristics. As a result, disparate sub-typing systems have emerged over the years to differentiate lineages of plague <span class="citation" data-cites="HqSIHWsh">[<a href="#ref-HqSIHWsh" role="doc-biblioref">13</a>]</span>. It has thus been argued that the taxonomy of <em>Y. pestis</em> should be revised and consolidated according to the latest global phylogenetic analysis <span class="citation" data-cites="S7uUqUQG">[<a href="#ref-S7uUqUQG" role="doc-biblioref">14</a>]</span>.</p>
<p>Unfortunately, there are a number of obstacles that have stalled large-scale phylogenetic analysis. The first challenge is data availability, both in terms of the genomic sequences, as well as the metadata required for interpretation. Genomic sampling of <em>Y. pestis</em> has recently intensified
<span class="citation" data-cites="E2sCnrTc">[<a href="#ref-E2sCnrTc" role="doc-biblioref">15</a>]</span>, thus providing exceptional new datasets for statistical inference. This intensive sampling has produced over 1000 <em>Y. pestis</em> genomes that are now publicly available, with tremendous diversity spanning five continents and 5000 years of human history. Unfortunately, the majority of these genomic records lack curated metadata, such as sampling date and location, which are crucial variables in testing population structure.</p>
<p>The second major obstacle is an apparent lack of temporal structure in <em>Y. pestis</em>. Detecting temporal signal and estimating a molecular clock model are general pre-requisites for sophisticated methods of quantifying population structure <span class="citation" data-cites="14eJbOlxZ">[<a href="#ref-14eJbOlxZ" role="doc-biblioref">16</a>]</span>.
However, there has been significant debate concerning whether <em>Y. pestis</em> can be appropriately modeled using the available clock methods <span class="citation" data-cites="ONhNS9aO ACt53Sow 1CPIgshmC">[<a href="#ref-ONhNS9aO" role="doc-biblioref">17</a>,<a href="#ref-ACt53Sow" role="doc-biblioref">18</a>,<a href="#ref-1CPIgshmC" role="doc-biblioref">19</a>]</span>.
To some extent, this debate can be explained by different <em>Y. pestis</em> datasets, which have been shown to produce dramatically different patterns of temporal signal <span class="citation" data-cites="1cTipQcd">[<a href="#ref-1cTipQcd" role="doc-biblioref">20</a>]</span>. Therefore, it is uncertain whether the new intensively sampled genomes will bring clarity or greater uncertainty.</p>
<p>In response to these debates and obstacles, this paper proposes a theoretical and methodological shift in plague genomics. Rather than conceptualizing <em>Y. pestis</em> as a conglomerate species, we highlight how novel insight emerges through analyzing <em>Y. pestis</em> sub-populations in isolation. To accomplish this shift in discourse, we focus on four objectives, specifically to:</p>
<ol type="1">
<li>Curate and contextualize the most recent <em>Y. pestis</em> genomic metadata.</li>
<li>Review and critique our current understanding of <em>Y. pestis</em> population structure.</li>
<li>Conduct robust and nuanced molecular clock analyses.</li>
<li>Identify key areas of phylogenetic uncertainty to be expanded on in future research.</li>
</ol>
<p>Progress towards these key objectives is anticipated to benefit both prospective studies of plague, such as environmental surveillance and outbreak monitoring, and retrospective studies, which seek to date emergence and spread of past pandemics.</p>
<h2 class="page_break_before" id="methods">Methods</h2>
<p>A visual overview of the computational methods is provided in Figure <a href="#fig:workflow">1</a> and is available as a snakemake pipeline (https://github.com/ktmeaton/plague-phylogeography/).</p>
<div id="fig:workflow" class="fignos">
<figure>
<img src="https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/6d9ab2e/main/report/workflow.png" alt="Figure 1: Computational methods workflow." /><figcaption aria-hidden="true"><span>Figure 1:</span> Computational methods workflow.</figcaption>
</figure>
</div>
<h3 class="page_break_before" id="data-collection">Data Collection</h3>
<p><em>Y. pestis</em> genome sequencing projects were retrieved from the NCBI databases using NCBImeta <span class="citation" data-cites="6YubepsW">[<a href="#ref-6YubepsW" role="doc-biblioref">21</a>]</span>. 1657 projects were identified and comprised three genomic types:</p>
<ul>
<li>586 modern assembled</li>
<li>184 ancient unassembled</li>
<li>887 modern unassembled</li>
</ul>
<p>The 887 modern unassembled genomes were excluded from this project, as the wide variety of laboratory methods and sequencing strategies precluded a standardized workflow. In contrast, the 184 ancient unassembled genomes were retained given the relatively standardized, albeit specialized, laboratory procedures required to process ancient tissues. Future work will investigate computationally efficient methods for integrating more diverse datasets.</p>
<p>Collection location, collection date, and collection host metadata were curated by cross-referencing the original publications. Collection location was transformed to latitude and longitude coordinates using GeoPy <span class="citation" data-cites="hxWSqZgm">[<a href="#ref-hxWSqZgm" role="doc-biblioref">22</a>]</span> and the Nominatim API <span class="citation" data-cites="M15a78x5">[<a href="#ref-M15a78x5" role="doc-biblioref">23</a>]</span> for OpenStreetMap <span class="citation" data-cites="5rCiNisz">[<a href="#ref-5rCiNisz" role="doc-biblioref">24</a>]</span>. Coordinates were standardized at a sub-country resolution, taking the centroid of the parent province/state. Collection dates were standardized according to their year, and recording uncertainty arising from missing data and radiocarbon estimates. Collection host was the most diverse field with regards to precision, ranging from colloquial nomenclature (<em>“rat”</em>) to a genus species taxonomy (<em>“Meriones libycus”</em>). For the purposes of this study, collection host was recorded as <em>Human</em>, <em>Non-Human</em>, or <em>Not Available</em>, given the inability to differentiate non-human mammalian hosts.</p>
<p>Genomes were removed if no associated date or location information could be identified in the literature, or if there was documented evidence of laboratory manipulation.</p>
<p>Two additional datasets were required for downstream analyses. First, <em>Y. pestis</em> strain CO92 (GCA_000009065.1) was used as the reference genome for sequence alignment and annotation. Second, <em>Yersinia pseudotuberculosis</em> strains NCTC10275 (GCA_900637475.1) and IP32953 (GCA_000834295.1) served as an outgroup to root the maximum likelihood phylogeny.</p>
<h3 id="alignment">Alignment</h3>
<p>Modern assembled genomes were aligned to the reference genome using Snippy, a pipeline for core genome alignments <span class="citation" data-cites="FysHdic6">[<a href="#ref-FysHdic6" role="doc-biblioref">25</a>]</span>. Modern genomes were removed if the number of sites covered at a minimum depth of 10X was less than 70% of the reference genome.</p>
<p>Ancient unassembled genomes were downloaded from the SRA database in FASTQ format using the SRA Toolkit <span class="citation" data-cites="puYDXtJ9">[<a href="#ref-puYDXtJ9" role="doc-biblioref">26</a>]</span>. Pre-processing and alignment to the reference genome was performed using the nf-core/eager pipeline, a reproducible workflow for ancient genome reconstruction <span class="citation" data-cites="17yD9OrGW">[<a href="#ref-17yD9OrGW" role="doc-biblioref">27</a>]</span>. Ancient genomes were removed if the number of sites covered at a minimum depth of 3X was less than 70% of the reference genome. It is a typical approach to relax coverage thresholds for ancient genomes relative to their modern counterparts <span class="citation" data-cites="cite">[<a href="#ref-cite" role="doc-biblioref"><strong>cite?</strong></a>]</span>. The threshold chosen here is commonly used, and aims to strike a balance between variant confidence and sample representation <span class="citation" data-cites="cite">[<a href="#ref-cite" role="doc-biblioref"><strong>cite?</strong></a>]</span>.</p>
<p>A multiple sequence alignment was constructed using the Snippy Core module of the Snippy pipeline <span class="citation" data-cites="FysHdic6">[<a href="#ref-FysHdic6" role="doc-biblioref">25</a>]</span>. The output alignment was filtered to only include chromosomal variants and to exclude sites that had more than 5% missing data.</p>
<h3 id="modified-datasets">Modified Datasets</h3>
<p>To investigate the influence of between-clade variation in substitution rates, the multiple alignment was separated into the major clades of <em>Y. pestis</em>, which is referred to as the <em>clade</em> dataset. Clade labeling was derived from the five-branch population structure accompanied by a biovar abbreviation <span class="citation" data-cites="ONhNS9aO">[<a href="#ref-ONhNS9aO" role="doc-biblioref">17</a>]</span>. Only one modification was made, in that the subclade associated with the Plague of Justinian (0.ANT4) was considered to be a distinct clade from its parent (0.ANT) due to its geographic, temporal, and ecological uniqueness.</p>
<p>To improve the performance and convergence of Bayesian analysis, a subsampled dataset was constructed and is referred to as the <em>reduced</em> dataset. Clades that contained multiple samples drawn from the same geographic location and the same time period were reduced to one representative sample. The sample with the shortest terminal branch length was prioritized, to diminish the influence of uniquely derived mutations on the estimated substitution rate. An interval of 25 years was identified as striking an optimal balance, resulting in 191 representative samples.</p>
<h3 id="phylogenetics">Phylogenetics</h3>
<p>Model selection was performed using Modelfinder which identified the K3Pu+F+I model as the optimal choice based on the Bayesian Information Criterion (BIC) <span class="citation" data-cites="QZIPWLUx">[<a href="#ref-QZIPWLUx" role="doc-biblioref">28</a>]</span>. A maximum-likelihood phylogeny was then estimated across 10 independent runs of IQTREE <span class="citation" data-cites="mkkgRhHT">[<a href="#ref-mkkgRhHT" role="doc-biblioref">29</a>]</span>. Branch support was evaluated using 1000 iterations of the ultrafast bootstrap approximation, with a threshold of 95% required for strong support <span class="citation" data-cites="12SvE6y3A">[<a href="#ref-12SvE6y3A" role="doc-biblioref">30</a>]</span>.</p>
<h3 id="phylodynamics">Phylodynamics</h3>
<p>To explore the degree of temporal signal present in the data, two categories of tests were performed . The first was a root-to-tip (RTT) regression on collection date using the python package <code>statsmodels</code> <span class="citation" data-cites="11e70fLkA">[<a href="#ref-11e70fLkA" role="doc-biblioref">31</a>]</span>. Given the relative simplicity of a regression model, the <em>full</em> dataset of 601 genomes was used. For the second test of temporal signal, a Bayesian Evaluation of Temporal Signal (BETS) was conducted. As the complexity of Bayesian modeling is computationally intensive, the <em>reduced</em> dataset (N=191) was used.</p>
<blockquote>
<p><strong>Kat’s Notes</strong>:<br>
- I will need Sebastian and Leo’s input here to write the BEAST methods section.</p>
</blockquote>
<h3 id="public-resource">Public Resource</h3>
<p>The maximum likelihood and Bayesian phylogenetic trees were uploaded to the Nextstrain visualization platform at https://nextstrain.org/community/ktmeaton/plague-phylogeography-projects@main. All curated metadata is available for download via Nextstrain, Github, and Zenodo.</p>
<blockquote>
<p><strong>Kat’s Notes</strong>:<br>
- I’ll need to prepare stable and archived hosting links before submission.</p>
</blockquote>
<h2 class="page_break_before" id="results-and-discussion">Results and Discussion</h2>
<h3 id="phylogeny">Phylogeny</h3>
<p>A maximum-likelihood phylogeny was estimated from 603 genomes (600 <em>Y. pestis</em> isolates, 1 <em>Y. pestis</em> reference, and 2 <em>Y. pseudotuberculosis</em> outgroup taxa). Following removal of the outgroup taxa, the alignment was composed of 10,249 variant positions with 3,844 sites shared by at least two genomes. In Figure <a href="#fig:divtree_branch-major">2</a>, the maximum-likeilhood phylogeny is visualized alongside the four major taxonomic systems currently used to define the population structure of <em>Y. pestis</em>. These include the major phylogenetic branches, biovars, subspecies, and historical time periods/pandemics.</p>
<div id="fig:divtree_branch-major" class="fignos">
<figure>
<img src="https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/e0542fd/main/auspice/all/chromosome/full/filter5/divtree_branch_major.png" alt="Figure 2: The maximum-likelihood phylogeny depicts the global population structure of Y. pestis. The divisions of the four major sub-typing systems are provided." /><figcaption aria-hidden="true"><span>Figure 2:</span> The maximum-likelihood phylogeny depicts the global population structure of <em>Y. pestis</em>. The divisions of the four major sub-typing systems are provided.</figcaption>
</figure>
</div>
<h3 id="population-structure">Population Structure</h3>
<p>A comparison of sub-typing systems reveals great uncertainty with regards to <em>Y. pestis</em> population structure. The global phylogeny of <em>Y. pestis</em> can be divided into <code>major branches</code> according to the relative position of the “big bang” polytomy <span class="citation" data-cites="ONhNS9aO">[<a href="#ref-ONhNS9aO" role="doc-biblioref">17</a>]</span>. All lineages that diverged prior to this multifurcation are grouped into <code>Branch 0</code> and those emerging after are the monophyletic clades <code>Branches 1-4</code>. Because the “big bang” polytomy plays such a central role in this system, there is growing interest in estimating its timing and geographic origins <span class="citation" data-cites="qLAL8Don">[<a href="#ref-qLAL8Don" role="doc-biblioref">32</a>]</span>. However, an inability to identify phenotypes distinguishing these <code>major branches</code> poses a significant challenge, and thus the exact epidemiological significance of the “big bang” polytomy remains unclear.</p>
<p>An example of this challenge can be seen in the population structure defined by the <code>biovar</code> system. <em>Y. pestis</em> can be categorized using a suite of metabolic properties into the classical biovars: <em>antiqua</em> (<code>ANT</code>), <em>medievalis</em> (<code>MED</code>), <em>orientalis</em> (<code>ORI</code>), and <em>microtus</em>/<em>pestoides</em> (<code>PE</code>) <span class="citation" data-cites="15WOrftK3 cIuFxbEY 66GmpEaG">[<a href="#ref-15WOrftK3" role="doc-biblioref">33</a>,<a href="#ref-cIuFxbEY" role="doc-biblioref">34</a>,<a href="#ref-66GmpEaG" role="doc-biblioref">35</a>]</span>. While these divisions don’t fully contradict the <code>major branches</code>, they do considerably shift the defining boundaries between <em>Y. pestis</em> populations.</p>
<p>To further complicate matters, researchers from the Commonwealth of Independent State (CIS) have observed biovar inconsistencies in plague foci surveillance <span class="citation" data-cites="S7uUqUQG">[<a href="#ref-S7uUqUQG" role="doc-biblioref">14</a>]</span>. In response, the <code>subspecies</code> taxonomy was created to distinguish a main subspecies (<em>pestis</em>) that is highly virulent in humans with wide geographic spread, from five or more non-main subspecies that have limited geographic ranges and attenuated virulence in humans <span class="citation" data-cites="S7uUqUQG">[<a href="#ref-S7uUqUQG" role="doc-biblioref">14</a>]</span>. The <code>subspecies</code> structure is highly convenient for laboratory diagnostics in the CIS, but struggles to differentiate the immense diversity represented by the larger <em>pestis</em> subspecies.</p>
<p>The challenge of categorizing plague by metabolism is, unsurprisingly, also an obstacle when analyzing extinct lineages. Ancient DNA (aDNA) researchers have opted to either extrapolate an existing biovar designation <span class="citation" data-cites="ACt53Sow">[<a href="#ref-ACt53Sow" role="doc-biblioref">18</a>]</span> or create a new one <span class="citation" data-cites="ZvyYZ3sx">[<a href="#ref-ZvyYZ3sx" role="doc-biblioref">9</a>]</span>. However, it is more common in aDNA studies to define population structure by time period and associations with historically documented pandemics. The known genetic diversity of <em>Y. pestis</em> thus far has been broadly grouped into four time periods and three pandemics (Figure <a href="#fig:timeline">3</a>).</p>
<div id="fig:timeline" class="fignos">
<figure>
<img src="https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/cd898b3/main/iqtree/all/chromosome/full/filter5/filter-taxa/timeline_all.png" alt="Figure 3: The temporal distribution of Y. pestis genomes." /><figcaption aria-hidden="true"><span>Figure 3:</span> The temporal distribution of <em>Y. pestis</em> genomes.</figcaption>
</figure>
</div>
<p>The First Pandemic (6th - 8th century CE) began with the Plague of Justinian and proceeded to devastate the Byzantine Empire of the Mediterranean world <span class="citation" data-cites="cite">[<a href="#ref-cite" role="doc-biblioref"><strong>cite?</strong></a>]</span>. The Late Antiquity clade found within <code>Branch 0</code> (<code>0.ANT4</code>) is associated with this pandemic given spatiotemporal overlap of the skeletal remains from which this lineage was retrieved <span class="citation" data-cites="ACt53Sow 19LMpiGZs">[<a href="#ref-ACt53Sow" role="doc-biblioref">18</a>,<a href="#ref-19LMpiGZs" role="doc-biblioref">36</a>]</span>. Similarly, the medieval clade <code>1.PRE</code> from <code>Branch 1</code> is thought to derive from the Second Pandemic of Plague. This well-documented pandemic began with the infamous Black Death and swept across most of Eurasia from the 14th to 19th centuries <span class="citation" data-cites="cite">[<a href="#ref-cite" role="doc-biblioref"><strong>cite?</strong></a>]</span>. The third documented pandemic of plague, alias the <em>Modern Pandemic</em>, spread globally from the end of the 18th Century and until the mid-20th Century. There is little dispute that a new lineage of plague emerging from Branch 1 as biovar <em>orientalis</em> (<code>1.ORI</code>) was the causative agent of this pandemic. While the World Health Organization (WHO) declared the third pandemic over in 1950 <span class="citation" data-cites="cite">[<a href="#ref-cite" role="doc-biblioref"><strong>cite?</strong></a>]</span>, this lineage continues to re-emerge to cause localized epidemics such as the 2010 plague in Peru <span class="citation" data-cites="cite">[<a href="#ref-cite" role="doc-biblioref"><strong>cite?</strong></a>]</span> and the Madagascar Outbreaks of 2017 <span class="citation" data-cites="cite">[<a href="#ref-cite" role="doc-biblioref"><strong>cite?</strong></a>]</span>.</p>
<p>While the pandemic clade nomenclature provides an excellent foundation for historical discussion, there are several emergent problems with this system. First is the growing awareness of the spatiotemporal overlap of the Second and the Third Pandemic <span class="citation" data-cites="cite">[<a href="#ref-cite" role="doc-biblioref"><strong>cite?</strong></a>]</span>. Previously, the temporal extents of these events were mutually exclusive, dating from the 14th-18th century, and the 19th-20th century respectively. Recent historical scholarship has contested this claim, and demonstrated that these constraints are a product of a Eurocentric view of plague <span class="citation" data-cites="cite">[<a href="#ref-cite" role="doc-biblioref"><strong>cite?</strong></a>]</span>. The Second Pandemic is now known to have extended into the 19th Century in parts of the Ottoman Empire, with the latest epidemics dating to 1819 <span class="citation" data-cites="cite">[<a href="#ref-cite" role="doc-biblioref"><strong>cite?</strong></a>]</span>. Similarly, the Third Pandemic is now hypothesized to have began as early as 1772 in southern China <span class="citation" data-cites="RWkOSyzX">[<a href="#ref-RWkOSyzX" role="doc-biblioref">37</a>]</span>. It remains unclear where to draw the distinction, if it even exists, between the Second and Third Pandemic.</p>
<p>Another limitation of the pandemic nomenclature is the complete disconnection of <code>Branch 2</code> to any historical pandemics. This is surprising given that several criteria of a pandemic pathogen are fulfilled by <code>Branch 2</code> lineages, namely extensive spread and virulence. <code>Branch 2</code> genomes of <em>Y. pestis</em> have been collected from all throughout Eurasia, stretching from at least the Caucasus, to India, and to eastern China (Figure <a href="#fig:map_branch2">4</a>). Furthermore, lineages of <code>Branch 2</code> have been associated with high mortality epidemics <span class="citation" data-cites="3aFEkVra">[<a href="#ref-3aFEkVra" role="doc-biblioref">38</a>]</span> and were observed to have the highest spread velocity of any <em>Y. pestis</em> clade <span class="citation" data-cites="RWkOSyzX">[<a href="#ref-RWkOSyzX" role="doc-biblioref">37</a>]</span> As historical plague scholarship extends beyond the bounds of Western Europe, it will be important to consider the role these lineages played, and adjust nomenclature accordingly.</p>
<div id="fig:map_branch2" class="fignos">
<figure>
<img src="https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/e0d84a1/main/auspice/all/chromosome/full/filter5/2_map.PNG" style="border:1px solid black" alt="Figure 4: The geographic distribution of Y. pestis Branch 2. (PLACEHOLDER)" /><figcaption aria-hidden="true"><span>Figure 4:</span> The geographic distribution of <em>Y. pestis</em> <code>Branch 2</code>. (PLACEHOLDER)</figcaption>
</figure>
</div>
<p>However, a significant obstacle to understanding the global spread and virulence of past plagues lies in <em>Y. pestis</em>’s weak host associations and a lack of geographic structure. Similar to its parent species, <em>Y. pseudotuberculosis</em> <span class="citation" data-cites="RrDa6yZx">[<a href="#ref-RrDa6yZx" role="doc-biblioref">11</a>]</span>, <em>Y. pestis</em> is capable of infecting a wide variety of mammalian hosts <span class="citation" data-cites="cite">[<a href="#ref-cite" role="doc-biblioref"><strong>cite?</strong></a>]</span>. But while isolates of <em>Y. pseudotuberculosis</em> cluster by host group <span class="citation" data-cites="MDriN15G">[<a href="#ref-MDriN15G" role="doc-biblioref">39</a>]</span>, the host structure of <em>Y. pestis</em> is cryptic (Figure <a href="#fig:divtree_host">5</a>). Most clades are isolated from both humans and non-human animals, although the ancient lineages <code>0.PRE</code>, (Bronze Age), <code>0.ANT4</code> (Late Antiquty) and <code>1.PRE</code> (Medieval/Early Modern) are exclusively associated with humans. However, this particular exception is largely due to sampling biases, as paleogenomic investigations have historically prioritized human skeletal remains over faunal remains <span class="citation" data-cites="cite">[<a href="#ref-cite" role="doc-biblioref"><strong>cite?</strong></a>]</span>.</p>
<div id="fig:divtree_host" class="fignos">
<figure>
<img src="https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/5af901d/main/auspice/all/chromosome/full/filter5/divtree_host.png" alt="Figure 5: The maximum-likelihood phylogeny of Y. pestis according to isolation host." /><figcaption aria-hidden="true"><span>Figure 5:</span> The maximum-likelihood phylogeny of <em>Y. pestis</em> according to isolation host.</figcaption>
</figure>
</div>
<p>In addition to a cryptic host structure, the geographic patterning of <em>Y. pestis</em>, or lack thereof, reflects a complex dispersal history (Figure <a href="#fig:map_all_branch_major">6</a>). Many regions have been colonized by diverse strains of <em>Y. pestis</em>. This diversity can be contemporaneous, such as endemic foci in the Caucausus and Western China (<code>0.PE</code>). Alternatively, this diversity may accrue over multiple centuries through distinct re-introductions and extinctions, as seen through historical clades in Europe (<code>0.ANT</code>, <code>1.PRE</code>). In these examples, a relatively large amount of genetic diversity appears in a small geographic range (Figure <a href="#fig:mantel_all_branch-major">7</a>). In contrast, regions such as the Americas have been colonized by a single strain of <em>Y. pestis</em> (<code>1.ORI</code>) which shows a relatively small amount of genetic diversity over a tremendously large geographic range.</p>
<p>This geographic complexity is unsurprising given that <em>Y. pseudotuberculosis</em>, the parent species of plague, also does not exhibit strong geographic structure. Outbreak strains of <em>Y. pseudotuberculosis</em> are particularly challenging to cluster, with non-outbreak lineages showing only slightly more geographic signal <span class="citation" data-cites="MDriN15G">[<a href="#ref-MDriN15G" role="doc-biblioref">39</a>]</span>. In this line of reasoning, the patterning observed in Figure <a href="#fig:map_all_branch_major">6</a> and <a href="#fig:mantel_all_branch-major">7</a> likely reflects the complex ecology of plague which cycles between endemic reservoirs and epidemic periods.</p>
<div id="fig:map_all_branch_major" class="fignos">
<figure>
<img src="https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/cd898b3/main/augur/all/chromosome/full/filter5/map_all.png" alt="Figure 6: The geographic distribution of Y. pestis genomes." /><figcaption aria-hidden="true"><span>Figure 6:</span> The geographic distribution of <em>Y. pestis</em> genomes.</figcaption>
</figure>
</div>
<div id="fig:mantel_all_branch-major" class="fignos">
<figure>
<img src="https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/cd898b3/main/iqtree/all/chromosome/full/filter5/filter-taxa/mantel_all.png" alt="Figure 7: Geographic distance vs. genetic distance. Statistical results come from a mantel test at α &lt;= 0.05." /><figcaption aria-hidden="true"><span>Figure 7:</span> Geographic distance vs. genetic distance. Statistical results come from a mantel test at α &lt;= 0.05.</figcaption>
</figure>
</div>
<p>However, it would be amiss to not acknowledge one of the largest caveats in genomic analyses, which is the issue of sampling bias. The geographic sampling strategy of <em>Y. pestis</em> genomes (Figure <a href="#fig:map_all_branch_major">6</a>) does not reflect the known distribution of modern plague <span class="citation" data-cites="pjjiQi2H">[<a href="#ref-pjjiQi2H" role="doc-biblioref">40</a>]</span>. The over-sampling of East Asia has been previously described by <span class="citation" data-cites="39QXTupk">[<a href="#ref-39QXTupk" role="doc-biblioref">41</a>]</span> and considerably drives the hypothesis that <em>Y. pestis</em> originated in China <span class="citation" data-cites="ONhNS9aO Y4CIJOZW">[<a href="#ref-ONhNS9aO" role="doc-biblioref">17</a>,<a href="#ref-Y4CIJOZW" role="doc-biblioref">42</a>]</span>. This once established hypothesis is now in contention, as the most basal strains of <em>Y. pestis</em> (Clades 0.PRE and 0.PE) have been isolated from all across Eurasia.</p>
<p>The sampling strategy of ancient DNA also does not reflect the hypothesized distribution of ancient plague. Historical genomes of <em>Y. pestis</em> have primarily been collected from European archaeological sites, with the most heavily sampled region being Western Europe (See Figure <span class="citation" data-cites="map_all_branch_major">[<a href="#ref-map_all_branch_major" role="doc-biblioref"><strong>map_all_branch_major?</strong></a>]</span>, 1.PRE).</p>
<blockquote>
<p><strong>Kat’s Notes:</strong><br>
- This section lacks an explicit conclusion and a transition to the next section.</p>
</blockquote>
<h3 class="page_break_before" id="phylodynamics-1">Phylodynamics</h3>
<h4 id="detecting-temporal-signal">Detecting Temporal Signal</h4>
<p>Previous work has documented substantial rate variation both between and within clades of <em>Y. pestis</em> <span class="citation" data-cites="ONhNS9aO 1CPIgshmC">[<a href="#ref-ONhNS9aO" role="doc-biblioref">17</a>,<a href="#ref-1CPIgshmC" role="doc-biblioref">19</a>]</span>. A root-to-tip regression on sampling date for the <em>full</em> dataset (N=601) reproduces this finding as the coefficient of determination (R<sup>2</sup>) is extremely low at 0.09 (Figure <a href="#fig:rtt_all_branch-major">8</a>). A Bayesian Evaluation of Temporal Signal (BETS) also indicates poor support for a strict clock as the coefficient of variation was consistently estimated to be greater than 1. Taken together, the root-to-tip regression and BETS analysis suggest that alternative clock models, such as the uncorrelated relaxed log normal (UCLN) model, should be preferred when accounting for the high degree of rate variation.</p>
<div id="fig:rtt_all_branch-major" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/ktmeaton/plague-phylogeography-projects/cd898b3/main/iqtree/all/chromosome/full/filter5/filter-taxa/rtt_all_branch_major.png" alt="Figure 8: Root-to-tip regression. The solid line represents the linear model for the entire dataset. The dashed lines present linear models for clades with significant p values." /><figcaption aria-hidden="true"><span>Figure 8:</span> Root-to-tip regression. The solid line represents the linear model for the entire dataset. The dashed lines present linear models for clades with significant p values.</figcaption>
</figure>
</div>
<p>However, the BETS analysis exhibited poor sampling of the relaxed clock model parameters, even when using a fixed topology (Figure <a href="#fig:mean_rate_trace_kde">9</a>). This suggests there may be too much rate variation to confidently estimate key parameters such as the mean substitution rate and the time to the most recent common ancestor (tMRCA). This observation is consistent with previous analyses <span class="citation" data-cites="ACt53Sow">[<a href="#ref-ACt53Sow" role="doc-biblioref">18</a>]</span> where robust estimates of model parameters could not be estimated, thus leading to the conclusion that <em>Y. pestis</em> lacks temporal signal. At the same time, other studies have suggested data composition is a strong determinant of temporal signal <span class="citation" data-cites="1cTipQcd">[<a href="#ref-1cTipQcd" role="doc-biblioref">20</a>]</span> and thus we investigated alternative approaches.</p>
<div id="fig:mean_rate_trace_kde" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/ktmeaton/plague-phylogeography-projects/c12ba4b/main/beast/all/chromosome/prune/filter5/mean_rate_trace_kde.png" alt="Figure 9: MCMC parameter estimation of the mean substitution rate for the reduced dataset (N=191). Left: Poor mixing of the MCMC Chain, Right: The resulting multimodal estimate of the rate." /><figcaption aria-hidden="true"><span>Figure 9:</span> MCMC parameter estimation of the mean substitution rate for the reduced dataset (N=191). Left: Poor mixing of the MCMC Chain, Right: The resulting multimodal estimate of the rate.</figcaption>
</figure>
</div>
<p>To identify patterns in rate variation that may improve the clock model, we first performed visual inspection of the root-to-tip regression residuals (Figure <a href="#fig:rtt_all_branch-major">8</a>). 3/12 clades appeared to have temporal signal according to a linear model, namely the ancient clades isolated from human skeletal remains: <code>0.PRE</code> (Bronze Age), <code>0.ANT4</code> (Late Antiquity), and <code>1.PRE</code> (Medieval/Early Modern). Indeed, when the root-to-tip regression was performed on clades in isolation, these three clades demonstrated strong evidence of strict-clock behavior (Table <a href="#tbl:rtt_statistics">3</a>, Figure <a href="#fig:rtt_clades">13</a>). A BETS analysis by clade proved even more sensitive as temporal signal was detected in 7/12 clades (Table <a href="#tbl:bets_temporal_signal">1</a>). Furthermore, for all clades with temporal signal, the relaxed clock model (UCLN) had higher support than the strict clock.</p>
<p>The ubiquitous support for a relaxed clock model was initially surprising, as the root-to-tip regression suggested strict clock-like behavior in several clades. However, this disparity can largely be explained by the known statistical limitations of a root-to-tip regression <span class="citation" data-cites="zikRADit">[<a href="#ref-zikRADit" role="doc-biblioref">43</a>]</span> which assumes either 1) no temporal structure, or 2) temporal structure following a linear model. Thus, a root-to-tip regression is strictly a test of the linear model, and will give no indication that other models are a better fit to the data, ie. a relaxed lognormal model. From this finding, we conclude that <strong>a root-to-tip regression is a poor statistical test of temporal signal in <em>Y. pestis</em>,</strong> and great caution should be taken in interpreting the associated statistics.</p>
<blockquote>
<p><strong>Kat’s Notes:</strong><br>
1. Could I get finalized Bayes Factors for the full dataset and clades?<br>
2. Could I get log files for all clades, even those without temporal signal?</p>
</blockquote>
<div id="tbl:bets_temporal_signal" class="tablenos">
<table id="tbl:bets_temporal_signal">
<caption><span>Table 1:</span> Temporal signal detection and clock model selection with Bayesian Evaluation of Temporal Signal (BETS) </caption>
<thead>
<tr class="header">
<th style="text-align: center;">Clade</th>
<th style="text-align: center;">N</th>
<th style="text-align: center;">SC Bayes Factor</th>
<th style="text-align: center;">UCLN Bayes Factor</th>
<th style="text-align: center;">Clock Model</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">1.ORI</td>
<td style="text-align: center;">117</td>
<td style="text-align: center;">29.6*</td>
<td style="text-align: center;">35.7*</td>
<td style="text-align: center;">UCLN</td>
</tr>
<tr class="even">
<td style="text-align: center;">1.IN</td>
<td style="text-align: center;">39</td>
<td style="text-align: center;">-3.9</td>
<td style="text-align: center;">-10.2</td>
<td style="text-align: center;">UCLN</td>
</tr>
<tr class="odd">
<td style="text-align: center;">1.ANT</td>
<td style="text-align: center;">4</td>
<td style="text-align: center;">8.9*</td>
<td style="text-align: center;">12.6*</td>
<td style="text-align: center;">UCLN</td>
</tr>
<tr class="even">
<td style="text-align: center;">1.PRE</td>
<td style="text-align: center;">40</td>
<td style="text-align: center;">10.1*</td>
<td style="text-align: center;">44.1*</td>
<td style="text-align: center;">UCLN</td>
</tr>
<tr class="odd">
<td style="text-align: center;">2.MED</td>
<td style="text-align: center;">116</td>
<td style="text-align: center;">TBD</td>
<td style="text-align: center;">TBD</td>
<td style="text-align: center;">TBD</td>
</tr>
<tr class="even">
<td style="text-align: center;">2.ANT</td>
<td style="text-align: center;">54</td>
<td style="text-align: center;">-20.8</td>
<td style="text-align: center;">-13.7</td>
<td style="text-align: center;">UCLN</td>
</tr>
<tr class="odd">
<td style="text-align: center;">4.ANT</td>
<td style="text-align: center;">11</td>
<td style="text-align: center;">-2.9</td>
<td style="text-align: center;">3.7*</td>
<td style="text-align: center;">UCLN</td>
</tr>
<tr class="even">
<td style="text-align: center;">3.ANT</td>
<td style="text-align: center;">11</td>
<td style="text-align: center;">-9.6</td>
<td style="text-align: center;">-11.4</td>
<td style="text-align: center;">UCLN</td>
</tr>
<tr class="odd">
<td style="text-align: center;">0.ANT</td>
<td style="text-align: center;">103</td>
<td style="text-align: center;">-2.3</td>
<td style="text-align: center;">-6.5</td>
<td style="text-align: center;">UCLN</td>
</tr>
<tr class="even">
<td style="text-align: center;">0.ANT4</td>
<td style="text-align: center;">12</td>
<td style="text-align: center;">5.3*</td>
<td style="text-align: center;">5.9*</td>
<td style="text-align: center;">UCLN</td>
</tr>
<tr class="odd">
<td style="text-align: center;">0.PE</td>
<td style="text-align: center;">83</td>
<td style="text-align: center;">-82.1</td>
<td style="text-align: center;">12.4*</td>
<td style="text-align: center;">UCLN</td>
</tr>
<tr class="even">
<td style="text-align: center;">0.PRE</td>
<td style="text-align: center;">8</td>
<td style="text-align: center;">TBD</td>
<td style="text-align: center;">TBD</td>
<td style="text-align: center;">TBD</td>
</tr>
</tbody>
</table>
</div>
<h4 id="rate-variation">Rate Variation</h4>
<p>Our approach of fitting nuanced models segregated by clade reveals that <strong>the ‘true’ substitution rates of <em>Y. pestis</em> may be much higher than previously thought.</strong> Previous work estimated that <em>Y. pestis</em> has one of the slowest observed substitution rates, around 1-2 x 10<sup>-8</sup>, which is on par with the exceptionally slow-evolving <em>Mycobacterium leprae</em> <span class="citation" data-cites="1cTipQcd ONhNS9aO">[<a href="#ref-1CPIgshmC" role="doc-biblioref">19</a>]</span>. The BETS analysis on the non-segregated data, which was highly unstable, fell within this published range with a 95% HPD between 1.16 x 10<sup>-8</sup> and 1.95 x 10<sup>-8</sup>. However, this global rate is a considerable underestimate, as clades with detectable temporal signal ranged from 2.33 x 10<sup>-8</sup> to 7.70 x 10<sup>-7</sup> (Table <a href="#tbl:bets_rate_cov_tmrca">2</a>, Figure <a href="#fig:substitution_rate_boxplot">10</a>).</p>
<blockquote>
<p><strong>Kat’s Notes:</strong><br>
- 0.PE appears to be an outlier, but wait for finalized logs.<br>
- No differences with regards to rate/variation between pandemic and non-pandemic clades.<br>
- I really want to see 1.IN, is there a progressive increase in rate along Branch 1?</p>
</blockquote>
<p>It is interesting to note that several clades have distinctly different substitution rates (ie.non-overlapping estimates of the mean substitution rate in Figure <a href="#fig:substitution_rate_boxplot">10</a>) and yet the relative amount of rate variation within a clade is similar (ie. overlapping coefficients of variation in Figure <a href="#fig:coefficient_of_variation_boxplot">11</a>).</p>
<p>We hypothesize that outlier clades which are challenging to model (ex. 2.MED) have artificially decreased estimates of the mean substitution rate in past studies. This study therefore reports the substitution rate of <em>Y. pestis</em> to be much higher than previously thought and more comparable to bacteria such as <em>Mycobacterium tuberulcosis</em>.</p>
<div id="tbl:bets_rate_cov_tmrca" class="tablenos">
<table id="tbl:bets_rate_cov_tmrca">
<caption><span>Table 2:</span> Estimate variation on the rate and tMRCA based on the 95% HPD. </caption>
<thead>
<tr class="header">
<th style="text-align: center;">Clade</th>
<th style="text-align: center;">N</th>
<th style="text-align: center;">Substitution Rate</th>
<th style="text-align: center;">Coefficient of Variation</th>
<th style="text-align: center;">tMRCA</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">All</td>
<td style="text-align: center;">191</td>
<td style="text-align: center;">1.16 x 10<sup>-8</sup> : 1.95 x 10<sup>-8</sup></td>
<td style="text-align: center;">1.35 : 2.10</td>
<td style="text-align: center;">-7400 : -3289</td>
</tr>
<tr class="even">
<td style="text-align: center;">1.ORI</td>
<td style="text-align: center;">117</td>
<td style="text-align: center;">1.04 x 10<sup>-7</sup> : 1.53 x 10<sup>-7</sup></td>
<td style="text-align: center;">1.02 : 1.87</td>
<td style="text-align: center;">1802 : 1907</td>
</tr>
<tr class="odd">
<td style="text-align: center;">1.IN</td>
<td style="text-align: center;">39</td>
<td style="text-align: center;">–</td>
<td style="text-align: center;">–</td>
<td style="text-align: center;">–</td>
</tr>
<tr class="even">
<td style="text-align: center;">1.ANT</td>
<td style="text-align: center;">4</td>
<td style="text-align: center;">3.61 x 10<sup>-8</sup> : 9.63 x 10<sup>-8</sup></td>
<td style="text-align: center;">0.00 : 2.12</td>
<td style="text-align: center;">1645 : 1889</td>
</tr>
<tr class="odd">
<td style="text-align: center;">1.PRE</td>
<td style="text-align: center;">40</td>
<td style="text-align: center;">3.69 x 10<sup>-8</sup> : 5.90 x 10<sup>-8</sup></td>
<td style="text-align: center;">1.22 : 3.24</td>
<td style="text-align: center;">995 : 1324</td>
</tr>
<tr class="even">
<td style="text-align: center;">2.MED</td>
<td style="text-align: center;">116</td>
<td style="text-align: center;">1.91 x 10<sup>-7</sup>, 3.01 x 10<sup>-7</sup></td>
<td style="text-align: center;">2.23 : 3.50</td>
<td style="text-align: center;">1547 : 1867</td>
</tr>
<tr class="odd">
<td style="text-align: center;">2.ANT</td>
<td style="text-align: center;">54</td>
<td style="text-align: center;">–</td>
<td style="text-align: center;">–</td>
<td style="text-align: center;">–</td>
</tr>
<tr class="even">
<td style="text-align: center;">4.ANT</td>
<td style="text-align: center;">11</td>
<td style="text-align: center;">3.59 x 10<sup>-8</sup> : 1.57 x 10<sup>-7</sup></td>
<td style="text-align: center;">1.22 : 3.59</td>
<td style="text-align: center;">1847 : 1975</td>
</tr>
<tr class="odd">
<td style="text-align: center;">3.ANT</td>
<td style="text-align: center;">11</td>
<td style="text-align: center;">–</td>
<td style="text-align: center;">–</td>
<td style="text-align: center;">–</td>
</tr>
<tr class="even">
<td style="text-align: center;">0.ANT</td>
<td style="text-align: center;">103</td>
<td style="text-align: center;">–</td>
<td style="text-align: center;">–</td>
<td style="text-align: center;">–</td>
</tr>
<tr class="odd">
<td style="text-align: center;">0.ANT4</td>
<td style="text-align: center;">12</td>
<td style="text-align: center;">2.33 x 10<sup>-8</sup> : 4.68 x 10<sup>-8</sup></td>
<td style="text-align: center;">1.40 : 3.94</td>
<td style="text-align: center;">39 : 237</td>
</tr>
<tr class="even">
<td style="text-align: center;">0.PE</td>
<td style="text-align: center;">83</td>
<td style="text-align: center;">4.32 x 10<sup>-7</sup> : 7.70 x 10<sup>-7</sup></td>
<td style="text-align: center;">1.70 : 2.78</td>
<td style="text-align: center;">1565 : 1892</td>
</tr>
<tr class="odd">
<td style="text-align: center;">0.PRE</td>
<td style="text-align: center;">8</td>
<td style="text-align: center;">3.95 x 10<sup>-8</sup> : 6.48 x 10<sup>-8</sup></td>
<td style="text-align: center;">0.84 : 3.13</td>
<td style="text-align: center;">-3070 : -2776</td>
</tr>
</tbody>
</table>
</div>
<div id="fig:substitution_rate_boxplot" class="fignos">
<figure>
<img src="https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/4a67df7c718b/main/beast/all/chromosome/clade/logs_noHyperPrior/substitution_rate.png" alt="Figure 10: Mean substitution rate uncertainty by clade." /><figcaption aria-hidden="true"><span>Figure 10:</span> Mean substitution rate uncertainty by clade.</figcaption>
</figure>
</div>
<div id="fig:coefficient_of_variation_boxplot" class="fignos">
<figure>
<img src="https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/4a67df7c718b/main/beast/all/chromosome/clade/logs_noHyperPrior/coefficient_of_variation.png" alt="Figure 11: Coefficient of variation uncertainty by clade." /><figcaption aria-hidden="true"><span>Figure 11:</span> Coefficient of variation uncertainty by clade.</figcaption>
</figure>
</div>
<h4 id="node-dating">Node Dating</h4>
<p>To evaluate the dates associated with ancestral events, we annotated the maximum likelihood phylogeny with the estimated tMRCAs for clades with temporal signal (Figure <a href="#fig:divtree_node_dating">12</a>). When re-contextualized into the global phylogeny, the node dates are topologically non-conflicting, meaning that parent nodes correctly pre-date child nodes. The sole exception is the estimated divergence of modern 0.PE which conflicts with the dates associated with the First and Second Pandemic clades. This conflict can be explained by several observations:</p>
<ol type="1">
<li>Clade 0.PE has the largest amount of uncertainty concerning the substitution rate.</li>
<li>Clade 0.PE has the greatest pairwise genetic distances, and longest root-to-tip distances to their MRCA.</li>
</ol>
<blockquote>
<ol type="1">
<li>The single ancient sample of 0.PE was excluded from the BEAST analysis! This was partly by mistake, as for other clades, I separated out the ancient and modern samples based on their drastically different sampling periods. But for 0.PE, there’s only one ancient sample and it got dropped.</li>
</ol>
</blockquote>
<div id="fig:divtree_node_dating" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/ktmeaton/plague-phylogeography-projects/206f481/main/auspice/all/chromosome/full/filter5/divtree_node-dating.png" alt="Figure 12: The maximum likelihood phylogeny annotated with the 95% HPD on clade root date." /><figcaption aria-hidden="true"><span>Figure 12:</span> The maximum likelihood phylogeny annotated with the 95% HPD on clade root date.</figcaption>
</figure>
</div>
<p>This drastic improvement in model performance reveals four intriguing aspects about the evolution of <em>Y. pestis</em>.</p>
<p>The first aspect is that <strong>fitting a single clock model to the global phylogeny of <em>Y. pestis</em> is not statistically supported</strong>. This can be observed in the relative instability of the MCMC analyses on the <em>reduced</em> dataset, which fails to converge in parameter space. In contrast, successfully fitting models on a clade-by-clade basis reveals that <strong><i>Y. pestis</i> has more temporal signal than previously thought.</strong> The observation that different populations have evolved at drastically different rates may explain the previous finding that the apparent structure in <em>Y. pestis</em> is dependent on dataset composition <span class="citation" data-cites="1cTipQcd">[<a href="#ref-1cTipQcd" role="doc-biblioref">20</a>]</span>.</p>
<p>The final finding from constructing nuanced models concerns the outlier clades for which no detectable signal could be found, namely 0.ANT, 2.ANT, 3.ANT, 2.MED, and 1.IN. The reason for this lack of signal is unclear, but one explanation may be that <strong>these <em>Y. pestis</em> populations are inappropriately separated based on the <code>major branch</code> and <code>biovar</code> systems</strong>. We hypothesis that alternative strategies to subdivide these populations will yield greater insight, based on the methodological improvements demonstrated in this study.</p>
<h2 class="page_break_before" id="conclusion">Conclusion</h2>
<blockquote>
<ol type="1">
<li><strong>Fitting a single clock model to the global phylogeny of <em>Y. pestis</em> is not statistically supported</strong>. <br> This can be observed in the relative instability of the MCMC analyses on the <em>reduced</em> dataset, which fails to converge in parameter space. <br></li>
<li><strong><em>Y. pestis</em> has much more temporal signal than previously thought.</strong><br>
Separating the genomic dataset by clade recovers robust temporal signal for the majority of clades.<br></li>
<li><strong>The true substitution rates of <em>Y. pestis</em> are much higher than previously thought.</strong><br>
The mean substitution rate of all global populations (1.59E-8) is a drastic underestimate compared to the rates observed by clade which range from 3.51E-8 to 1.29E-7. The clades without temporal signal are pulling down the mean estimate. Previous work estimated that <em>Y. pestis</em> has one of the lowest observed substitution rates, on par with the exceptionally slow-evolving <em>Mycobacterium leprae</em> <span class="citation" data-cites="1cTipQcd">[<a href="#ref-1cTipQcd" role="doc-biblioref">20</a>]</span>. This study instead reports the substitution rate of <em>Y. pestis</em> to be much higher, and comparable to <em>Mycobacterium tuberulcosis</em>.<br></li>
<li><strong>Root-to-tip regression is a poor statistical test of temporal signal compared to BETS</strong>.<br>
The root-to-tip regression has several known limitations, namely the underlying assumption of strict clock-behavior and the non-independence of data points <span class="citation" data-cites="zikRADit">[<a href="#ref-zikRADit" role="doc-biblioref">43</a>]</span>. A BETS analysis counters these statistical violates, and is overall more sensitive given that multiple clock models can be tested. In this study, root-to-tip regression indicated temporal signal in 3/12 lineages while the BETS analysis detected signal in 7/12 lineages.<br></li>
</ol>
</blockquote>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-Vo6ReJPm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline"><strong>The Stone Age Plague and Its Persistence in Eurasia</strong> <div class="csl-block">Aida Andrades Valtueña, Alissa Mittnik, Felix M Key, Wolfgang Haak, Raili Allmäe, Andrej Belinskij, Mantas Daubaras, Michal Feldman, Rimantas Jankauskas, Ivor Janković, … Johannes Krause</div> <em>Current Biology</em> (2017-12-04) <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cub.2017.10.025">10.1016/j.cub.2017.10.025</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29174893">29174893</a></div></div>
</div>
<div id="ref-AQa9Tn4j" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline"><strong>Emergence and spread of basal lineages of &lt;i&gt;Yersinia pestis&lt;/i&gt; during the Neolithic Decline</strong> <div class="csl-block">Nicolás Rascovan, Karl-Göran Sjögren, Kristian Kristiansen, Rasmus Nielsen, Eske Willerslev, Christelle Desnues, Simon Rasmussen</div> <em>Cell</em> (2019-01-10) <a href="https://www.cell.com/cell/abstract/S0092-8674(18)31464-8">https://www.cell.com/cell/abstract/S0092-8674(18)31464-8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cell.2018.11.005">10.1016/j.cell.2018.11.005</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30528431">30528431</a></div></div>
</div>
<div id="ref-1093vihdz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline"><strong>Trade routes and plague transmission in pre-industrial Europe</strong> <div class="csl-block">Ricci PH Yue, Harry F Lee, Connor YH Wu</div> <em>Scientific Reports</em> (2017-10-11) <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5636801/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5636801/</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41598-017-13481-2">10.1038/s41598-017-13481-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29021541">29021541</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5636801">PMC5636801</a></div></div>
</div>
<div id="ref-uM6Rh5Fu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline"><strong>Yersinia pestis--etiologic agent of plague</strong> <div class="csl-block">RD Perry, JD Fetherston</div> <em>Clinical Microbiology Reviews</em> (1997-01) <div class="csl-block">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8993858">8993858</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC172914">PMC172914</a></div></div>
</div>
<div id="ref-h4WAqbKy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline"><strong>Plague</strong> <div class="csl-block">World Health Organization</div> (2017-10-31) <a href="https://www.who.int/news-room/fact-sheets/detail/plague">https://www.who.int/news-room/fact-sheets/detail/plague</a></div>
</div>
<div id="ref-13ET92iS0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline"><strong>The Black Death, 1346-1353: The Complete History</strong> <div class="csl-block">OJ Benedictow</div> <em>Boydell Press</em> (2004) <div class="csl-block">ISBN: <a href="https://worldcat.org/isbn/0-85115-943-5">0-85115-943-5</a></div></div>
</div>
<div id="ref-Nd9WUU1j" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline"><strong>Plague around the world in 2019</strong> <div class="csl-block">Eric Bertherat</div> <em>Weekly Epidemiological Record</em> (2019-06-21) <a href="https://apps.who.int/iris/bitstream/handle/10665/325481/WER9425-en-fr.pdf">https://apps.who.int/iris/bitstream/handle/10665/325481/WER9425-en-fr.pdf</a></div>
</div>
<div id="ref-kQFNGkNf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline"><strong>Recent trends in plague ecology</strong> <div class="csl-block">K Gage, M Kosoy</div> (2006) <a href="http://reviverestore.org/wp-content/uploads/2015/02/Gage-and-Kosoy_USGS-Blk-footed-ferret-symp_2006-copy.pdf">http://reviverestore.org/wp-content/uploads/2015/02/Gage-and-Kosoy_USGS-Blk-footed-ferret-symp_2006-copy.pdf</a></div>
</div>
<div id="ref-ZvyYZ3sx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline"><strong>Early Divergent Strains of Yersinia pestis in Eurasia 5,000 Years Ago</strong> <div class="csl-block">Simon Rasmussen, Morten Erik Allentoft, Kasper Nielsen, Ludovic Orlando, Martin Sikora, Karl-Göran Sjögren, Anders Gorm Pedersen, Mikkel Schubert, Alex Van Dam, Christian Moliin Outzen Kapel, … Eske Willerslev</div> <em>Cell</em> (2015-10-22) <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4644222/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4644222/</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cell.2015.10.009">10.1016/j.cell.2015.10.009</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26496604">26496604</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4644222">PMC4644222</a></div></div>
</div>
<div id="ref-TV8iDhk7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline"><strong>Yersinia pestis, the cause of plague, is a recently emerged clone of Yersinia pseudotuberculosis</strong> <div class="csl-block">M Achtman, K Zurth, G Morelli, G Torrea, A Guiyoule, E Carniel</div> <em>Proceedings of the National Academy of Sciences of the United States of America</em> (1999-11-23) <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.96.24.14043">10.1073/pnas.96.24.14043</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10570195">10570195</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC24187">PMC24187</a></div></div>
</div>
<div id="ref-RrDa6yZx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline"><strong>Population structure of the Yersinia pseudotuberculosis complex according to multilocus sequence typing</strong> <div class="csl-block">Riikka Laukkanen-Ninios, Xavier Didelot, Keith A Jolley, Giovanna Morelli, Vartul Sangal, Paula Kristo, Priscilla FM Imori, Hiroshi Fukushima, Anja Siitonen, Galina Tseneva, … Mark Achtman</div> <em>Environmental microbiology</em> (2011-12) <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3988354/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3988354/</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1111/j.1462-2920.2011.02588.x">10.1111/j.1462-2920.2011.02588.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21951486">21951486</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3988354">PMC3988354</a></div></div>
</div>
<div id="ref-17ygZQMtK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline"><strong>Phylogeographic separation and formation of sexually discrete lineages in a global population of Yersinia pseudotuberculosis</strong> <div class="csl-block">Tristan Seecharran, Laura Kalin-Manttari, Katja Koskela, Simo Nikkari, Benjamin Dickins, Jukka Corander, Mikael Skurnik, AlanYR 2017 McNally</div> <em>Microbial Genomics</em> (2017) <a href="https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000133">https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000133</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1099/mgen.0.000133">10.1099/mgen.0.000133</a></div></div>
</div>
<div id="ref-HqSIHWsh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline"><strong>Taxonomy of Yersinia pestis</strong> <div class="csl-block">Zhizhen Qi, Yujun Cui, Qingwen Zhang, Ruifu Yang</div> <em>Yersinia pestis: Retrospective and Perspective</em> (2016) <a href="https://doi.org/10.1007/978-94-024-0890-4_3">https://doi.org/10.1007/978-94-024-0890-4_3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-94-024-0890-4_3">10.1007/978-94-024-0890-4_3</a> · ISBN: <a href="https://worldcat.org/isbn/978-94-024-0890-4">978-94-024-0890-4</a></div></div>
</div>
<div id="ref-S7uUqUQG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline"><strong>Phylogeny and classification of &lt;i&gt;Yersinia pestis&lt;/i&gt; through the lens of strains From the plague foci of Commonwealth of Independent States</strong> <div class="csl-block">Vladimir V Kutyrev, Galina A Eroshenko, Vladimir L Motin, Nikita Y Nosov, Jaroslav M Krasnov, Lyubov M Kukleva, Konstantin A Nikiforov, Zhanna V Al’khova, Eugene G Oglodin, Natalia P Guseva</div> <em>Frontiers in Microbiology</em> (2018-05-25) <a href="https://www.frontiersin.org/article/10.3389/fmicb.2018.01106/full">https://www.frontiersin.org/article/10.3389/fmicb.2018.01106/full</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3389/fmicb.2018.01106">10.3389/fmicb.2018.01106</a></div></div>
</div>
<div id="ref-E2sCnrTc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline"><strong>The EnteroBase user's guide, with case studies on &lt;i&gt;Salmonella&lt;/i&gt; transmissions, &lt;i&gt;Yersinia pestis&lt;/i&gt; phylogeny, and &lt;i&gt;Escherichia&lt;/i&gt; core genomic diversity</strong> <div class="csl-block">Zhemin Zhou, Nabil-Fareed Alikhan, Khaled Mohamed, Yulei Fan, Mark Achtman</div> <em>Genome Research</em> (2020-01) <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6961584/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6961584/</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/gr.251678.119">10.1101/gr.251678.119</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31809257">31809257</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6961584">PMC6961584</a></div></div>
</div>
<div id="ref-14eJbOlxZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline"><strong>Identification of Hidden Population Structure in Time-Scaled Phylogenies</strong> <div class="csl-block">Erik M Volz, Wiuf Carsten, Yonatan H Grad, Simon DW Frost, Ann M Dennis, Xavier Didelot</div> <em>Systematic Biology</em> (2020-09-01) <a href="https://doi.org/10.1093/sysbio/syaa009">https://doi.org/10.1093/sysbio/syaa009</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/sysbio/syaa009">10.1093/sysbio/syaa009</a></div></div>
</div>
<div id="ref-ONhNS9aO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline"><strong>Historical variations in mutation rate in an epidemic pathogen, &lt;i&gt;Yersinia pestis&lt;/i&gt;</strong> <div class="csl-block">Y Cui, C Yu, Y Yan, D Li, Y Li, T Jombart, LA Weinert, Z Wang, Z Guo, L Xu, … R Yang</div> <em>Proceedings of the National Academy of Sciences</em> (2013-01-08) <a href="http://www.pnas.org/cgi/doi/10.1073/pnas.1205750110">http://www.pnas.org/cgi/doi/10.1073/pnas.1205750110</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.1205750110">10.1073/pnas.1205750110</a></div></div>
</div>
<div id="ref-ACt53Sow" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline"><strong>&lt;i&gt;Yersinia pestis&lt;/i&gt; and the Plague of Justinian 541–543 AD: a genomic analysis</strong> <div class="csl-block">David M Wagner, Jennifer Klunk, Michaela Harbeck, Alison Devault, Nicholas Waglechner, Jason W Sahl, Jacob Enk, Dawn N Birdsell, Melanie Kuch, Candice Lumibao, … Hendrik Poinar</div> <em>The Lancet Infectious Diseases</em> (2014-04) <a href="https://linkinghub.elsevier.com/retrieve/pii/S1473309913703232">https://linkinghub.elsevier.com/retrieve/pii/S1473309913703232</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/S1473-3099(13)70323-2">10.1016/s1473-3099(13)70323-2</a></div></div>
</div>
<div id="ref-1CPIgshmC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline"><strong>Phylogeography of the second plague pandemic revealed through analysis of historical Yersinia pestis genomes</strong> <div class="csl-block">Maria A Spyrou, Marcel Keller, Rezeda I Tukhbatova, Christiana L Scheib, Elizabeth A Nelson, Aida Andrades Valtueña, Gunnar U Neumann, Don Walker, Amelie Alterauge, Niamh Carty, … Johannes Krause</div> <em>Nature Communications</em> (2019-10-02) <a href="https://www.nature.com/articles/s41467-019-12154-0">https://www.nature.com/articles/s41467-019-12154-0</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-019-12154-0">10.1038/s41467-019-12154-0</a></div></div>
</div>
<div id="ref-1cTipQcd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline"><strong>Genome-scale rates of evolutionary change in bacteria</strong> <div class="csl-block">Sebastian Duchêne, Kathryn E Holt, François-Xavier Weill, Simon Le Hello, Jane Hawkey, David J Edwards, Mathieu Fourment, Edward C Holmes</div> <em>Microbial Genomics</em> (2016-11-30) <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5320706/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5320706/</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1099/mgen.0.000094">10.1099/mgen.0.000094</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28348834">28348834</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5320706">PMC5320706</a></div></div>
</div>
<div id="ref-6YubepsW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline"><strong>NCBImeta: efficient and comprehensive metadata retrieval from NCBI databases</strong> <div class="csl-block">Katherine Eaton</div> <em>Journal of Open Source Software</em> (2020) <a href="https://doi.org/10.21105/joss.01990">https://doi.org/10.21105/joss.01990</a> <div class="csl-block">DOI: <a href="https://doi.org/10.21105/joss.01990">10.21105/joss.01990</a></div></div>
</div>
<div id="ref-hxWSqZgm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline"><strong>GeoPy: A Python client for several popular geocoding web services.</strong> <div class="csl-block">Kostya Esmukov</div> (2020-12) <a href="https://github.com/geopy/geopy">https://github.com/geopy/geopy</a></div>
</div>
<div id="ref-M15a78x5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">23. </div><div class="csl-right-inline"><strong>Nominatim: A tool to search OpenStreetMap data.</strong> <div class="csl-block">Sarah Hoffman</div> (2020-12) <a href="https://github.com/osm-search/Nominatim">https://github.com/osm-search/Nominatim</a></div>
</div>
<div id="ref-5rCiNisz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">24. </div><div class="csl-right-inline"><strong>Planet dump retrieved from https://planet.osm.org</strong> <div class="csl-block">OpenStreetMap Contributors</div> (2017) <a href="https://www.openstreetmap.org">https://www.openstreetmap.org</a></div>
</div>
<div id="ref-FysHdic6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">25. </div><div class="csl-right-inline"><strong>Snippy: Rapid haploid variant calling and core genome alignment.</strong> <div class="csl-block">Torsten Seemann</div> (2020-03-08) <a href="https://github.com/tseemann/snippy">https://github.com/tseemann/snippy</a></div>
</div>
<div id="ref-puYDXtJ9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">26. </div><div class="csl-right-inline"><strong>ncbi/sra-tools</strong> <div class="csl-block">NCBI - National Center for Biotechnology Information/NLM/NIH</div> (2021-05-18) <a href="https://github.com/ncbi/sra-tools">https://github.com/ncbi/sra-tools</a></div>
</div>
<div id="ref-17yD9OrGW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">27. </div><div class="csl-right-inline"><strong>Reproducible, portable, and efficient ancient genome reconstruction with nf-core/eager</strong> <div class="csl-block">James AFellows Yates, Thiseas C Lamnidis, Maxime Borry, Aida Andrades Valtueña, Zandra Fagernäs, Stephen Clayton, Maxime U Garcia, Judith Neukamm, Alexander Peltzer</div> <em>PeerJ</em> (2021-03-16) <a href="https://peerj.com/articles/10947">https://peerj.com/articles/10947</a> <div class="csl-block">DOI: <a href="https://doi.org/10.7717/peerj.10947">10.7717/peerj.10947</a></div></div>
</div>
<div id="ref-QZIPWLUx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">28. </div><div class="csl-right-inline"><strong>ModelFinder: fast model selection for accurate phylogenetic estimates</strong> <div class="csl-block">Subha Kalyaanamoorthy, Bui Quang Minh, Thomas KF Wong, Arndt von Haeseler, Lars S Jermiin</div> <em>Nature Methods</em> (2017-06) <a href="http://www.nature.com/articles/nmeth.4285">http://www.nature.com/articles/nmeth.4285</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.4285">10.1038/nmeth.4285</a></div></div>
</div>
<div id="ref-mkkgRhHT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">29. </div><div class="csl-right-inline"><strong>IQ-TREE 2: New Models and Efficient Methods for Phylogenetic Inference in the Genomic Era</strong> <div class="csl-block">Bui Quang Minh, Heiko A Schmidt, Olga Chernomor, Dominik Schrempf, Michael D Woodhams, Arndt von Haeseler, Robert Lanfear</div> <em>Molecular Biology and Evolution</em> (2020-05-01) <a href="https://academic.oup.com/mbe/article/37/5/1530/5721363">https://academic.oup.com/mbe/article/37/5/1530/5721363</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/molbev/msaa015">10.1093/molbev/msaa015</a></div></div>
</div>
<div id="ref-12SvE6y3A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">30. </div><div class="csl-right-inline"><strong>UFBoot2: Improving the Ultrafast Bootstrap Approximation</strong> <div class="csl-block">Diep Thi Hoang, Olga Chernomor, Arndt von Haeseler, Bui Quang Minh, Le Sy Vinh</div> <em>Molecular Biology and Evolution</em> (2018-02-01) <a href="https://academic.oup.com/mbe/article/35/2/518/4565479">https://academic.oup.com/mbe/article/35/2/518/4565479</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/molbev/msx281">10.1093/molbev/msx281</a></div></div>
</div>
<div id="ref-11e70fLkA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">31. </div><div class="csl-right-inline"><strong>Statsmodels: Econometric and Statistical Modeling with Python</strong> <div class="csl-block">Skipper Seabold, Josef Perktold</div> (2010) <a href="https://conference.scipy.org/proceedings/scipy2010/seabold.html">https://conference.scipy.org/proceedings/scipy2010/seabold.html</a> <div class="csl-block">DOI: <a href="https://doi.org/10.25080/Majora-92bf1922-011">10.25080/majora-92bf1922-011</a></div></div>
</div>
<div id="ref-qLAL8Don" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">32. </div><div class="csl-right-inline"><strong>How a microbe becomes a pandemic: a new story of the Black Death</strong> <div class="csl-block">Monica H Green</div> <em>The Lancet Microbe</em> (2020-12-01) <a href="https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247(20)30176-2/abstract">https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247(20)30176-2/abstract</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/S2666-5247(20)30176-2">10.1016/s2666-5247(20)30176-2</a></div></div>
</div>
<div id="ref-15WOrftK3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">33. </div><div class="csl-right-inline"><strong>Variétés de l'espèce Pasteurella pestis</strong> <div class="csl-block">R Devignat</div> <em>Bulletin of the World Health Organization</em> (1951) <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2554099/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2554099/</a> <div class="csl-block">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14859080">14859080</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2554099">PMC2554099</a></div></div>
</div>
<div id="ref-cIuFxbEY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">34. </div><div class="csl-right-inline"><strong>Comparative and evolutionary genomics of &lt;i&gt;Yersinia pestis&lt;/i&gt;</strong> <div class="csl-block">Dongsheng Zhou, Yanping Han, Yajun Song, Peitang Huang, Ruifu Yang</div> <em>Microbes and Infection</em> (2004-11-01) <a href="http://www.sciencedirect.com/science/article/pii/S1286457904002357">http://www.sciencedirect.com/science/article/pii/S1286457904002357</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.micinf.2004.08.002">10.1016/j.micinf.2004.08.002</a></div></div>
</div>
<div id="ref-66GmpEaG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">35. </div><div class="csl-right-inline"><strong>Genotyping and Phylogenetic Analysis of Yersinia pestis by MLVA: Insights into the Worldwide Expansion of Central Asia Plague Foci</strong> <div class="csl-block">Yanjun Li, Yujun Cui, Yolande Hauck, Mikhail E Platonov, Erhei Dai, Yajun Song, Zhaobiao Guo, Christine Pourcel, Svetlana V Dentovskaya, Andrey P Anisimov, … Gilles Vergnaud</div> <em>PLOS ONE</em> (2009-06-22) <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0006000">https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0006000</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pone.0006000">10.1371/journal.pone.0006000</a></div></div>
</div>
<div id="ref-19LMpiGZs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">36. </div><div class="csl-right-inline"><strong>Ancient &lt;i&gt;Yersinia pestis&lt;/i&gt; genomes from across Western Europe reveal early diversification during the First Pandemic (541–750)</strong> <div class="csl-block">Marcel Keller, Maria A Spyrou, Christiana L Scheib, Gunnar U Neumann, Andreas Kröpelin, Brigitte Haas-Gebhard, Bernd Päffgen, Jochen Haberstroh, Albert Ribera i Lacomba, Claude Raynaud, … Johannes Krause</div> <em>Proceedings of the National Academy of Sciences</em> (2019-06-18) <a href="https://www.pnas.org/content/116/25/12363">https://www.pnas.org/content/116/25/12363</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.1820447116">10.1073/pnas.1820447116</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31164419">31164419</a></div></div>
</div>
<div id="ref-RWkOSyzX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">37. </div><div class="csl-right-inline"><strong>Wet climate and transportation routes accelerate spread of human plague</strong> <div class="csl-block">Lei Xu, Leif Chr Stige, Kyrre Linné Kausrud, Tamara Ben Ari, Shuchun Wang, Xiye Fang, Boris V Schmid, Qiyong Liu, Nils Chr Stenseth, Zhibin Zhang</div> <em>Proceedings of the Royal Society B: Biological Sciences</em> (2014-04-07) <a href="https://royalsocietypublishing.org/doi/10.1098/rspb.2013.3159">https://royalsocietypublishing.org/doi/10.1098/rspb.2013.3159</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1098/rspb.2013.3159">10.1098/rspb.2013.3159</a></div></div>
</div>
<div id="ref-3aFEkVra" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">38. </div><div class="csl-right-inline"><strong>Evolution and circulation of Yersinia pestis in the Northern Caspian and Northern Aral Sea regions in the 20th-21st centuries</strong> <div class="csl-block">Galina A Eroshenko, Nikolay V Popov, Zhanna V Al’khova, Lyubov M Kukleva, Alina N Balykova, Nadezhda S Chervyakova, Ekaterina A Naryshkina, Vladimir V Kutyrev</div> <em>PLOS ONE</em> (2021-02-11) <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0244615">https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0244615</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pone.0244615">10.1371/journal.pone.0244615</a></div></div>
</div>
<div id="ref-MDriN15G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">39. </div><div class="csl-right-inline"><strong>Genomic Insights into a Sustained National Outbreak of Yersinia pseudotuberculosis</strong> <div class="csl-block">Deborah A Williamson, Sarah L Baines, Glen P Carter, Anders Gonçalves da Silva, Xiaoyun Ren, Jill Sherwood, Muriel Dufour, Mark B Schultz, Nigel P French, Torsten Seemann, … Benjamin P Howden</div> <em>Genome Biology and Evolution</em> (2016-12-01) <a href="https://doi.org/10.1093/gbe/evw285">https://doi.org/10.1093/gbe/evw285</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/gbe/evw285">10.1093/gbe/evw285</a></div></div>
</div>
<div id="ref-pjjiQi2H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">40. </div><div class="csl-right-inline"><strong>Historical and genomic data reveal the influencing factors on global transmission velocity of plague during the Third Pandemic</strong> <div class="csl-block">Lei Xu, Leif C Stige, Herwig Leirs, Simon Neerinckx, Kenneth L Gage, Ruifu Yang, Qiyong Liu, Barbara Bramanti, Katharine R Dean, Hui Tang, … Zhibin Zhang</div> <em>Proceedings of the National Academy of Sciences</em> (2019-06-11) <a href="https://www.pnas.org/content/116/24/11833">https://www.pnas.org/content/116/24/11833</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.1901366116">10.1073/pnas.1901366116</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31138696">31138696</a></div></div>
</div>
<div id="ref-39QXTupk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">41. </div><div class="csl-right-inline"><strong>Historical &lt;i&gt;Y. pestis&lt;/i&gt; Genomes Reveal the European Black Death as the Source of Ancient and Modern Plague Pandemics</strong> <div class="csl-block">Maria A Spyrou, Rezeda I Tukhbatova, Michal Feldman, Joanna Drath, Sacha Kacki, Julia Beltrán de Heredia, Susanne Arnold, Airat G Sitdikov, Dominique Castex, Joachim Wahl, … Johannes Krause</div> <em>Cell Host &amp; Microbe</em> (2016-06) <a href="http://linkinghub.elsevier.com/retrieve/pii/S1931312816302086">http://linkinghub.elsevier.com/retrieve/pii/S1931312816302086</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.chom.2016.05.012">10.1016/j.chom.2016.05.012</a></div></div>
</div>
<div id="ref-Y4CIJOZW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">42. </div><div class="csl-right-inline"><strong>&lt;i&gt;Yersinia pestis&lt;/i&gt; genome sequencing identifies patterns of global phylogenetic diversity</strong> <div class="csl-block">Giovanna Morelli, Yajun Song, Camila J Mazzoni, Mark Eppinger, Philippe Roumagnac, David M Wagner, Mirjam Feldkamp, Barica Kusecek, Amy J Vogler, Yanjun Li, … Mark Achtman</div> <em>Nature Genetics</em> (2010-12) <div class="csl-block">DOI: <a href="https://doi.org/10.1038/ng.705">10.1038/ng.705</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21037571">21037571</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2999892">PMC2999892</a></div></div>
</div>
<div id="ref-zikRADit" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">43. </div><div class="csl-right-inline"><strong>Bayesian Evaluation of Temporal Signal in Measurably Evolving Populations</strong> <div class="csl-block">Sebastian Duchene, Philippe Lemey, Tanja Stadler, Simon YW Ho, David A Duchene, Vijaykrishna Dhanasekaran, Guy Baele</div> <em>Molecular Biology and Evolution</em> (2020-11-01) <a href="https://doi.org/10.1093/molbev/msaa163">https://doi.org/10.1093/molbev/msaa163</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/molbev/msaa163">10.1093/molbev/msaa163</a></div></div>
</div>
</div>
<h2 class="page_break_before" id="appendix">Appendix</h2>
<h3 id="phylodynamics-2">Phylodynamics</h3>
<h4 id="root-to-tip-regression">Root to Tip Regression</h4>
<div id="tbl:rtt_statistics" class="tablenos">
<table id="tbl:rtt_statistics">
<caption><span>Table 3:</span> Root-to-tip regression statistics. </caption>
<thead>
<tr class="header">
<th style="text-align: center;">Clade</th>
<th style="text-align: center;">N</th>
<th style="text-align: center;">Coefficient</th>
<th style="text-align: center;">p</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">All</td>
<td style="text-align: center;">601</td>
<td style="text-align: center;">0.09</td>
<td style="text-align: center;">4.959e-13*</td>
</tr>
<tr class="even">
<td style="text-align: center;">1.ORI</td>
<td style="text-align: center;">117</td>
<td style="text-align: center;">0.04</td>
<td style="text-align: center;">1.184e-01</td>
</tr>
<tr class="odd">
<td style="text-align: center;">1.IN</td>
<td style="text-align: center;">39</td>
<td style="text-align: center;">-0.0</td>
<td style="text-align: center;">1.0</td>
</tr>
<tr class="even">
<td style="text-align: center;">1.ANT</td>
<td style="text-align: center;">4</td>
<td style="text-align: center;">0.45</td>
<td style="text-align: center;">1.0</td>
</tr>
<tr class="odd">
<td style="text-align: center;">1.PRE</td>
<td style="text-align: center;">40</td>
<td style="text-align: center;">0.76</td>
<td style="text-align: center;">2.022e-12*</td>
</tr>
<tr class="even">
<td style="text-align: center;">2.MED</td>
<td style="text-align: center;">116</td>
<td style="text-align: center;">0.01</td>
<td style="text-align: center;">1.0</td>
</tr>
<tr class="odd">
<td style="text-align: center;">2.ANT</td>
<td style="text-align: center;">54</td>
<td style="text-align: center;">0.05</td>
<td style="text-align: center;">4.764e-01</td>
</tr>
<tr class="even">
<td style="text-align: center;">4.ANT</td>
<td style="text-align: center;">11</td>
<td style="text-align: center;">-0.11</td>
<td style="text-align: center;">1.0</td>
</tr>
<tr class="odd">
<td style="text-align: center;">3.ANT</td>
<td style="text-align: center;">11</td>
<td style="text-align: center;">-0.04</td>
<td style="text-align: center;">1.0</td>
</tr>
<tr class="even">
<td style="text-align: center;">0.ANT</td>
<td style="text-align: center;">103</td>
<td style="text-align: center;">-0.01</td>
<td style="text-align: center;">1.0</td>
</tr>
<tr class="odd">
<td style="text-align: center;">0.ANT4</td>
<td style="text-align: center;">12</td>
<td style="text-align: center;">0.66</td>
<td style="text-align: center;">7.842e-03*</td>
</tr>
<tr class="even">
<td style="text-align: center;">0.PE</td>
<td style="text-align: center;">83</td>
<td style="text-align: center;">0.01</td>
<td style="text-align: center;">1.0</td>
</tr>
<tr class="odd">
<td style="text-align: center;">0.PRE</td>
<td style="text-align: center;">8</td>
<td style="text-align: center;">0.91</td>
<td style="text-align: center;">1.678e-03*</td>
</tr>
</tbody>
</table>
</div>
<div id="fig:rtt_clades" class="fignos">
<figure>
<img src="https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/cd898b3/main/iqtree/all/chromosome/full/filter5/filter-taxa/rtt_clades.png" alt="Figure 13: Root To Tip Regression by clade." /><figcaption aria-hidden="true"><span>Figure 13:</span> Root To Tip Regression by clade.</figcaption>
</figure>
</div>
<h3 id="geography">Geography</h3>
<div id="fig:mantel_clades" class="fignos">
<figure>
<img src="https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/56f5c21cc83a62bb02132a433992fcfbbe13cbb6/main/iqtree/all/chromosome/full/filter5/filter-taxa/mantel_clades.png" alt="Figure 14: Mantel test by clade." /><figcaption aria-hidden="true"><span>Figure 14:</span> Mantel test by clade.</figcaption>
</figure>
</div>
<!-- default theme -->

<style>
  /* import google fonts */
  @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
  @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

  /* -------------------------------------------------- */
  /* global */
  /* -------------------------------------------------- */

  /* all elements */
  * {
    /* force sans-serif font unless specified otherwise */
    font-family: "Open Sans", "Helvetica", sans-serif;

    /* prevent text inflation on some mobile browsers */
    -webkit-text-size-adjust: none !important;
    -moz-text-size-adjust: none !important;
    -o-text-size-adjust: none !important;
    text-size-adjust: none !important;
  }

  @media only screen {
    /* "page" element */
    body {
      position: relative;
      box-sizing: border-box;
      font-size: 12pt;
      line-height: 1.5;
      max-width: 8.5in;
      margin: 20px auto;
      padding: 40px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* "page" element */
    body {
      padding: 20px;
      margin: 0;
      border-radius: 0;
      border: none;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
      background: none;
    }
  }

  /* -------------------------------------------------- */
  /* headings */
  /* -------------------------------------------------- */

  /* all headings */
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 20px 0;
    padding: 0;
    font-weight: bold;
  }

  /* biggest heading */
  h1 {
    margin: 40px 0;
    text-align: center;
  }

  /* second biggest heading */
  h2 {
    margin-top: 30px;
    padding-bottom: 5px;
    border-bottom: solid 1px #bdbdbd;
  }

  /* heading font sizes */
  h1 {
    font-size: 2em;
  }
  h2 {
    font-size: 1.5em;
  }
  h3 {
    font-size: 1.35em;
  }
  h4 {
    font-size: 1.25em;
  }
  h5 {
    font-size: 1.15em;
  }
  h6 {
    font-size: 1em;
  }

  /* -------------------------------------------------- */
  /* manuscript header */
  /* -------------------------------------------------- */

  /* manuscript title */
  header > h1 {
    margin: 0;
  }

  /* manuscript title caption text (ie "automatically generated on") */
  header + p {
    text-align: center;
    margin-top: 10px;
  }

  /* -------------------------------------------------- */
  /* text elements */
  /* -------------------------------------------------- */

  /* links */
  a {
    color: #2196f3;
    overflow-wrap: break-word;
  }

  /* superscripts and subscripts */
  sub,
  sup {
    /* prevent from affecting line height */
    line-height: 0;
  }

  /* unordered and ordered lists*/
  ul,
  ol {
    padding-left: 20px;
  }

  /* class for styling text semibold */
  .semibold {
    font-weight: 600;
  }

  /* class for styling elements horizontally left aligned */
  .left {
    display: block;
    text-align: left;
    margin-left: auto;
    margin-right: 0;
    justify-content: left;
  }

  /* class for styling elements horizontally centered */
  .center {
    display: block;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    justify-content: center;
  }

  /* class for styling elements horizontally right aligned */
  .right {
    display: block;
    text-align: right;
    margin-left: 0;
    margin-right: auto;
    justify-content: right;
  }

  /* -------------------------------------------------- */
  /* section elements */
  /* -------------------------------------------------- */

  /* horizontal divider line */
  hr {
    border: none;
    height: 1px;
    background: #bdbdbd;
  }

  /* paragraphs, horizontal dividers, figures, tables, code */
  p,
  hr,
  figure,
  table,
  pre {
    /* treat all as "paragraphs", with consistent vertical margins */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* figures */
  /* -------------------------------------------------- */

  /* figure */
  figure {
    max-width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure caption */
  figcaption {
    padding: 0;
    padding-top: 10px;
  }

  /* figure image element */
  figure > img,
  figure > svg {
    max-width: 100%;
    display: block;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure auto-number */
  img + figcaption > span:first-of-type,
  svg + figcaption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* tables */
  /* -------------------------------------------------- */

  /* table */
  table {
    border-collapse: collapse;
    border-spacing: 0;
    width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* table cells */
  th,
  td {
    border: solid 1px #bdbdbd;
    padding: 10px;
    /* squash table if too wide for page by forcing line breaks */
    overflow-wrap: break-word;
    word-break: break-word;
  }

  /* header row and even rows */
  th,
  tr:nth-child(2n) {
    background-color: #fafafa;
  }

  /* odd rows */
  tr:nth-child(2n + 1) {
    background-color: #ffffff;
  }

  /* table caption */
  caption {
    text-align: left;
    padding: 0;
    padding-bottom: 10px;
  }

  /* table auto-number */
  table > caption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* code */
  /* -------------------------------------------------- */

  /* multi-line code block */
  pre {
    padding: 10px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
    break-inside: avoid;
    text-align: left;
  }

  /* inline code, ie code within normal text */
  :not(pre) > code {
    padding: 0 4px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
  }

  /* code text */
  /* apply all children, to reach syntax highlighting sub-elements */
  code,
  code * {
    /* force monospace font */
    font-family: "Source Code Pro", "Courier New", monospace;
  }

  /* -------------------------------------------------- */
  /* quotes */
  /* -------------------------------------------------- */

  /* quoted text */
  blockquote {
    margin: 0;
    padding: 0;
    border-left: 4px solid #bdbdbd;
    padding-left: 16px;
    break-inside: avoid;
  }

  /* -------------------------------------------------- */
  /* banners */
  /* -------------------------------------------------- */

  /* info banners */
  .banner {
    box-sizing: border-box;
    display: block;
    position: relative;
    width: 100%;
    margin-top: 20px;
    margin-bottom: 20px;
    padding: 20px;
    text-align: center;
  }

  /* paragraph in banner */
  .banner > p {
    margin: 0;
  }

  /* -------------------------------------------------- */
  /* highlight colors */
  /* -------------------------------------------------- */

  .white {
    background: #ffffff;
  }
  .lightgrey {
    background: #eeeeee;
  }
  .grey {
    background: #757575;
  }
  .darkgrey {
    background: #424242;
  }
  .black {
    background: #000000;
  }
  .lightred {
    background: #ffcdd2;
  }
  .lightyellow {
    background: #ffecb3;
  }
  .lightgreen {
    background: #dcedc8;
  }
  .lightblue {
    background: #e3f2fd;
  }
  .lightpurple {
    background: #f3e5f5;
  }
  .red {
    background: #f44336;
  }
  .orange {
    background: #ff9800;
  }
  .yellow {
    background: #ffeb3b;
  }
  .green {
    background: #4caf50;
  }
  .blue {
    background: #2196f3;
  }
  .purple {
    background: #9c27b0;
  }
  .white,
  .lightgrey,
  .lightred,
  .lightyellow,
  .lightgreen,
  .lightblue,
  .lightpurple,
  .orange,
  .yellow,
  .white a,
  .lightgrey a,
  .lightred a,
  .lightyellow a,
  .lightgreen a,
  .lightblue a,
  .lightpurple a,
  .orange a,
  .yellow a {
    color: #000000;
  }
  .grey,
  .darkgrey,
  .black,
  .red,
  .green,
  .blue,
  .purple,
  .grey a,
  .darkgrey a,
  .black a,
  .red a,
  .green a,
  .blue a,
  .purple a {
    color: #ffffff;
  }

  /* -------------------------------------------------- */
  /* buttons */
  /* -------------------------------------------------- */

  /* class for styling links like buttons */
  .button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    margin: 5px;
    padding: 10px 20px;
    font-size: 0.75em;
    font-weight: 600;
    text-transform: uppercase;
    text-decoration: none;
    letter-spacing: 1px;
    background: none;
    color: #2196f3;
    border: solid 1px #bdbdbd;
    border-radius: 5px;
  }

  /* buttons when hovered */
  .button:hover:not([disabled]),
  .icon_button:hover:not([disabled]) {
    cursor: pointer;
    background: #f5f5f5;
  }

  /* buttons when disabled */
  .button[disabled],
  .icon_button[disabled] {
    opacity: 0.35;
    pointer-events: none;
  }

  /* class for styling buttons containg only single icon */
  .icon_button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    margin: 0;
    padding: 0;
    background: none;
    border-radius: 5px;
    border: none;
    width: 20px;
    height: 20px;
    min-width: 20px;
    min-height: 20px;
  }

  /* icon button inner svg image */
  .icon_button > svg {
    height: 16px;
  }

  /* -------------------------------------------------- */
  /* icons */
  /* -------------------------------------------------- */

  /* class for styling icons inline with text */
  .inline_icon {
    height: 1em;
    position: relative;
    top: 0.125em;
  }

  /* -------------------------------------------------- */
  /* references */
  /* -------------------------------------------------- */

  .csl-entry {
    margin-top: 15px;
    margin-bottom: 15px;
  }

  /* -------------------------------------------------- */
  /* print control */
  /* -------------------------------------------------- */

  @media print {
    @page {
      /* suggested printing margin */
      margin: 0.5in;
    }

    /* document and "page" elements */
    html,
    body {
      margin: 0;
      padding: 0;
      width: 100%;
      height: 100%;
    }

    /* "page" element */
    body {
      font-size: 11pt !important;
      line-height: 1.35;
    }

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
      margin: 15px 0;
    }

    /* figures and tables */
    figure,
    table {
      font-size: 0.85em;
    }

    /* table cells */
    th,
    td {
      padding: 5px;
    }

    /* shrink font awesome icons */
    i.fas,
    i.fab,
    i.far,
    i.fal {
      transform: scale(0.85);
    }

    /* decrease banner margins */
    .banner {
      margin-top: 15px;
      margin-bottom: 15px;
      padding: 15px;
    }

    /* class for centering an element vertically on its own page */
    .page_center {
      margin: auto;
      width: 100%;
      height: 100%;
      display: flex;
      align-items: center;
      vertical-align: middle;
      break-before: page;
      break-after: page;
    }

    /* always insert a page break before the element */
    .page_break_before {
      break-before: page;
    }

    /* always insert a page break after the element */
    .page_break_after {
      break-after: page;
    }

    /* avoid page break before the element */
    .page_break_before_avoid {
      break-before: avoid;
    }

    /* avoid page break after the element */
    .page_break_after_avoid {
      break-after: avoid;
    }

    /* avoid page break inside the element */
    .page_break_inside_avoid {
      break-inside: avoid;
    }
  }

  /* -------------------------------------------------- */
  /* override pandoc css quirks */
  /* -------------------------------------------------- */

  .sourceCode {
    /* prevent unsightly overflow in wide code blocks */
    overflow: auto !important;
  }

  div.sourceCode {
    /* prevent background fill on top-most code block  container */
    background: none !important;
  }

  .sourceCode * {
    /* force consistent line spacing */
    line-height: 1.5 !important;
  }

  div.sourceCode {
    /* style code block margins same as <pre> element */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* tablenos */
  /* -------------------------------------------------- */

  /* tablenos wrapper */
  .tablenos {
    width: 100%;
    margin: 20px 0;
  }

  .tablenos > table {
    /* move margins from table to table_wrapper to allow margin collapsing */
    margin: 0;
  }

  @media only screen {
    /* tablenos wrapper */
    .tablenos {
      /* show scrollbar on tables if necessary to prevent overflow */
      overflow-x: auto !important;
    }

    .tablenos th,
    .tablenos td {
      overflow-wrap: unset !important;
      word-break: unset !important;
    }

    /* table in wrapper */
    .tablenos table,
    .tablenos table * {
      /* don't break table words */
      overflow-wrap: normal !important;
    }
  }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen())
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery =
    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false")
      makeButtons(`div[id^="ref-"]`, ".csl-left-margin", "reference");
    if (figures !== "false")
      makeButtons(`div[id^="fig:"]`, "figcaption", "figure");
    if (tables !== "false") makeButtons(`div[id^="tbl:"]`, "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(query);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
      "class",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML =
      document.querySelector(".icon_caret_right").innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector(":nth-child(2)");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script>
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
</body>
</html>
