
## Results and Discussion  {.page_break_before}

### Phylogeny

A maximum-likelihood phylogeny was estimated from 603 genomes (600 *Y. pestis* isolates,  1 *Y. pestis* reference, and 2 *Y. pseudotuberculosis* outgroup taxa). Following removal of the outgroup taxa, the alignment was composed of 10,249 variant positions with 3,844 sites shared by at least two genomes. In Figure @fig:divtree_branch-major, the maximum-likeilhood phylogeny is visualized alongside the four major taxonomic systems currently used to define the population structure of *Y. pestis*. These include the major phylogenetic branches, biovars, subspecies, and historical time periods/pandemics.

![
The maximum-likelihood phylogeny depicts the global population structure of *Y. pestis*. The divisions of the four major sub-typing systems are provided.
](https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/e0542fd/main/auspice/all/chromosome/full/filter5/divtree_branch_major.png){#fig:divtree_branch-major}

### Population Structure

A comparison of sub-typing systems reveals great uncertainty with regards to *Y. pestis* population structure. The global phylogeny of *Y. pestis* can be divided into ```major branches``` according to the relative position of the "big bang" polytomy [@cui2013HistoricalVariationsMutation]. All lineages that diverged prior to this multifurcation are grouped into ```Branch 0``` and those emerging after are the monophyletic clades ```Branches 1-4```. Because the "big bang" polytomy plays such a central role in this system, there is growing interest in estimating its timing and geographic origins [@green2020HowMicrobeBecomes]. However, an inability to identify phenotypes distinguishing these ```major branches``` poses a significant challenge, and thus the exact epidemiological significance of the "big bang" polytomy remains unclear.

An example of this challenge can be seen in the population structure defined by the ```biovar``` system. *Y. pestis* can be categorized using a suite of metabolic properties into the classical biovars: *antiqua* (```ANT```), *medievalis* (```MED```), *orientalis* (```ORI```), and *microtus*/*pestoides* (```PE```) [@devignat1951VarietesEspecePasteurella; @zhou2004ComparativeEvolutionaryGenomics; @li2009GenotypingPhylogeneticAnalysis]. While these divisions don't fully contradict the ```major branches```, they do considerably shift the defining boundaries between *Y. pestis* populations.

To further complicate matters, researchers from the Commonwealth of Independent State (CIS) have observed biovar inconsistencies in plague foci surveillance [@kutyrev2018PhylogenyClassificationYersinia]. In response, the ```subspecies``` taxonomy was created to distinguish a main subspecies (*pestis*) that is highly virulent in humans with wide geographic spread, from five or more non-main subspecies that have limited geographic ranges and attenuated virulence in humans [@kutyrev2018PhylogenyClassificationYersinia]. The ```subspecies``` structure is highly convenient for laboratory diagnostics in the CIS, but struggles to differentiate the immense diversity represented by the larger *pestis* subspecies.

The challenge of categorizing plague by metabolism is, unsurprisingly, also an obstacle when analyzing extinct lineages. Ancient DNA (aDNA) researchers have opted to either extrapolate an existing biovar designation [@wagner2014YersiniaPestisPlague] or create a new one [@rasmussen2015EarlyDivergentStrains]. However, it is more common in aDNA studies to define population structure by time period and associations with historically documented pandemics. The known genetic diversity of *Y. pestis* thus far has been broadly grouped into four time periods and three pandemics (Figure @fig:timeline).

![
The temporal distribution of *Y. pestis* genomes.
](https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/cd898b3/main/iqtree/all/chromosome/full/filter5/filter-taxa/timeline_all.png){#fig:timeline}

The First Pandemic (6th - 8th century CE) began with the Plague of Justinian and proceeded to devastate the Byzantine Empire of the Mediterranean world [@cite]. The Late Antiquity clade found within ```Branch 0``` (```0.ANT4```) is associated with this pandemic given spatiotemporal overlap of the skeletal remains from which this lineage was retrieved [@wagner2014YersiniaPestisPlague; @keller2019AncientYersiniaPestis].  Similarly, the medieval clade ```1.PRE``` from ```Branch 1``` is thought to derive from the Second Pandemic of Plague. This well-documented pandemic began with the infamous Black Death and swept across most of Eurasia from the 14th to 19th centuries [@cite]. The third documented pandemic of plague, alias the *Modern Pandemic*, spread globally from the end of the 18th Century and until the mid-20th Century. There is little dispute that a new lineage of plague emerging from Branch 1 as biovar *orientalis* (```1.ORI```) was the causative agent of this pandemic. While the World Health Organization (WHO) declared the third pandemic over in 1950 [@cite], this lineage continues to re-emerge to cause localized epidemics such as the 2010 plague in Peru [@cite] and the Madagascar Outbreaks of 2017 [@cite].

While the pandemic clade nomenclature provides an excellent foundation for historical discussion, there are several emergent problems with this system. First is the growing awareness of the spatiotemporal overlap of the Second and the Third Pandemic [@cite]. Previously, the temporal extents of these events were mutually exclusive, dating from the 14th-18th century, and the 19th-20th century respectively. Recent historical scholarship has contested this claim, and demonstrated that these constraints are a product of a Eurocentric view of plague [@cite]. The Second Pandemic is now known to have extended into the 19th Century in parts of the Ottoman Empire, with the latest epidemics dating to 1819 [@cite]. Similarly, the Third Pandemic is now hypothesized to have began as early as 1772 in southern China [@xu2014WetClimateTransportation]. It remains unclear where to draw the distinction, if it even exists, between the Second and Third Pandemic.

Another limitation of the pandemic nomenclature is the complete disconnection of ```Branch 2``` to any historical pandemics. This is surprising given that several criteria of a pandemic pathogen are fulfilled by ```Branch 2``` lineages, namely extensive spread and virulence. ```Branch 2``` genomes of *Y. pestis* have been collected from all throughout Eurasia, stretching from at least the Caucasus, to India, and to eastern China (Figure @fig:map_branch2). Furthermore, lineages of ```Branch 2``` have been associated with high mortality epidemics [@eroshenko2021EvolutionCirculationYersinia] and were observed to have the highest spread velocity of any *Y. pestis* clade [@xu2014WetClimateTransportation] As historical plague scholarship extends beyond the bounds of Western Europe, it will be important to consider the role these lineages played, and adjust nomenclature accordingly.

![The geographic distribution of *Y. pestis* ```Branch 2```. (PLACEHOLDER)](https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/e0d84a1/main/auspice/all/chromosome/full/filter5/2_map.PNG){#fig:map_branch2 style="border:1px solid black"}

However, a significant obstacle to understanding the global spread and virulence of past plagues lies in *Y. pestis*'s weak host associations and a lack of geographic structure. Similar to its parent species, *Y. pseudotuberculosis*  [@laukkanen-ninios2011PopulationStructureYersinia], *Y. pestis* is capable of infecting a wide variety of mammalian hosts [@cite]. But while isolates of *Y. pseudotuberculosis* cluster by host group [@williamson2016GenomicInsightsSustained], the host structure of *Y. pestis* is cryptic (Figure @fig:divtree_host). Most clades are isolated from both humans and non-human animals, although the ancient lineages ```0.PRE```, (Bronze Age), ```0.ANT4``` (Late Antiquty) and ```1.PRE``` (Medieval/Early Modern) are exclusively associated with humans. However, this particular exception is largely due to sampling biases, as paleogenomic investigations have historically prioritized human skeletal remains over faunal remains [@cite].

![
The maximum-likelihood phylogeny of *Y. pestis* according to isolation host.
](https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/5af901d/main/auspice/all/chromosome/full/filter5/divtree_host.png){#fig:divtree_host}


In addition to a cryptic host structure, the geographic patterning of *Y. pestis*, or lack thereof, reflects a complex dispersal history (Figure @fig:map_all_branch_major). Many regions have been colonized by diverse strains of *Y. pestis*. This diversity can be contemporaneous, such as endemic foci in the Caucausus and Western China (```0.PE```). Alternatively, this diversity may accrue over multiple centuries through distinct re-introductions and extinctions, as seen through historical clades in Europe (```0.ANT```, ```1.PRE```). In these examples, a relatively large amount of genetic diversity appears in a small geographic range (Figure @fig:mantel_all_branch-major). In contrast, regions such as the Americas have been colonized by a single strain of *Y. pestis* (```1.ORI```) which shows a relatively small amount of genetic diversity over a tremendously large geographic range.

This geographic complexity is unsurprising given that *Y. pseudotuberculosis*, the parent species of plague, also does not exhibit strong geographic structure. Outbreak strains of *Y. pseudotuberculosis* are particularly challenging to cluster, with non-outbreak lineages showing only slightly more geographic signal [@williamson2016GenomicInsightsSustained]. In this line of reasoning, the patterning observed in Figure @fig:map_all_branch_major and @fig:mantel_all_branch-major likely reflects the complex ecology of plague which cycles between endemic reservoirs and epidemic periods.

![
The geographic distribution of *Y. pestis* genomes.
](https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/cd898b3/main/augur/all/chromosome/full/filter5/map_all.png){#fig:map_all_branch_major}

![
Geographic distance vs. genetic distance. Statistical results come from a mantel test at α <= 0.05.
](https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/cd898b3/main/iqtree/all/chromosome/full/filter5/filter-taxa/mantel_all.png){#fig:mantel_all_branch-major}


However, it would be amiss to not acknowledge one of the largest caveats in genomic analyses, which is the issue of sampling bias. The geographic sampling strategy of *Y. pestis* genomes (Figure @fig:map_all_branch_major) does not reflect the known distribution of modern plague [@xu2019HistoricalGenomicData]. The over-sampling of East Asia has been previously described by @spyrou2016HistoricalPestisGenomes and considerably drives the hypothesis that *Y. pestis* originated in China [@cui2013HistoricalVariationsMutation; @morelli2010YersiniaPestisGenome]. This once established hypothesis is now in contention, as the most basal strains of *Y. pestis* (Clades 0.PRE and 0.PE) have been isolated from all across Eurasia.

The sampling strategy of ancient DNA also does not reflect the hypothesized distribution of ancient plague. Historical genomes of *Y. pestis* have primarily been collected from European archaeological sites, with the most heavily sampled region being Western Europe (See Figure @map_all_branch_major, 1.PRE).

> **Kat's Notes:**<br>
> - This section lacks an explicit conclusion and a transition to the next section.

### Phylodynamics {.page_break_before}

#### Detecting Temporal Signal

Previous work has documented substantial rate variation both between and within clades of *Y. pestis* [@cui2013HistoricalVariationsMutation; @spyrou2019PhylogeographySecondPlague]. A root-to-tip regression on sampling date for the *full* dataset (N=601) reproduces this finding as the coefficient of determination (R<sup>2</sup>) is extremely low at 0.09 (Figure @fig:rtt_all_branch-major). A  Bayesian Evaluation of Temporal Signal (BETS) also indicates poor support for a strict clock as the coefficient of variation was consistently estimated to be greater than 1. Taken together, the root-to-tip regression and BETS analysis suggest that alternative clock models, such as the uncorrelated relaxed log normal (UCLN) model, should be preferred when accounting for the high degree of rate variation.

![ Root-to-tip regression. The solid line represents the linear model for the entire dataset. The dashed lines present linear models for clades with significant p values.](https://raw.githubusercontent.com/ktmeaton/plague-phylogeography-projects/cd898b3/main/iqtree/all/chromosome/full/filter5/filter-taxa/rtt_all_branch_major.png){#fig:rtt_all_branch-major}

However, the BETS analysis exhibited poor sampling of the relaxed clock model parameters, even when using a fixed topology (Figure @fig:mean_rate_trace_kde). This suggests there may be too much rate variation to confidently estimate key parameters such as the mean substitution rate and the time to the most recent common ancestor (tMRCA). This observation is consistent with previous analyses [@wagner2014YersiniaPestisPlague] where robust estimates of model parameters could not be estimated, thus leading to the conclusion that *Y. pestis* lacks temporal signal. At the same time, other studies have suggested data composition is a strong determinant of temporal signal @duchene2016GenomescaleRatesEvolutionary and thus we investigated alternative approaches.

![MCMC parameter estimation of the mean substitution rate for the reduced dataset (N=191). Left: Poor mixing of the MCMC Chain, Right: The resulting multimodal estimate of the rate.](https://raw.githubusercontent.com/ktmeaton/plague-phylogeography-projects/c12ba4b/main/beast/all/chromosome/prune/filter5/mean_rate_trace_kde.png){#fig:mean_rate_trace_kde}

To identify patterns in rate variation that may improve the clock model, we first performed visual inspection of the root-to-tip regression residuals (Figure @fig:rtt_all_branch-major). 3/12 clades appeared to have temporal signal according to a linear model, namely the ancient clades isolated from human skeletal remains: ```0.PRE``` (Bronze Age),  ```0.ANT4``` (Late Antiquity), and ```1.PRE``` (Medieval/Early Modern). Indeed, when the root-to-tip regression was performed on clades in isolation, these three clades demonstrated strong evidence of strict-clock behavior (Table @tbl:rtt_statistics, Figure @fig:rtt_clades). A BETS analysis by clade proved even more sensitive as temporal signal was detected in 7/12 clades (Table @tbl:bets_temporal_signal). Furthermore, for all clades with temporal signal, the relaxed clock model (UCLN) had higher support than the strict clock.

The ubiquitous support for a relaxed clock model was initially surprising, as the root-to-tip regression suggested strict clock-like behavior in several clades. However, this disparity can largely be explained by the known statistical limitations of a root-to-tip regression [@duchene2020BayesianEvaluationTemporal] which assumes either 1) no temporal structure, or 2) temporal structure following a linear model. Thus, a root-to-tip regression is strictly a test of the linear model, and will give no indication that other models are a better fit to the data, ie. a relaxed lognormal model. From this finding, we conclude that **a root-to-tip regression is a poor statistical test of temporal signal in *Y. pestis*,** and great caution should be taken in interpreting the associated statistics.


> **Kat's Notes:**<br>
> 1. Could I get finalized Bayes Factors for the full dataset and clades?<br>
> 2. Could I get log files for all clades, even those without temporal signal?

|   Clade    |  N  | SC Bayes Factor | UCLN Bayes Factor | Clock Model |
|:----------:|:---:|:---------------:|:-----------------:|:-----------:|
| 1.ORI  | 117 |      29.6*      |       35.7*       |    UCLN     |
|  1.IN  | 39  |      -3.9       |       -10.2       |    UCLN     |
| 1.ANT  |  4  |      8.9*       |       12.6*       |    UCLN     |
| 1.PRE  | 40  |      10.1*      |       44.1*       |    UCLN     |
| 2.MED  | 116 |       TBD       |        TBD        |     TBD     |
| 2.ANT  | 54  |      -20.8      |       -13.7       |    UCLN     |
| 4.ANT  | 11  |      -2.9       |       3.7*        |    UCLN     |
| 3.ANT  | 11  |      -9.6       |       -11.4       |    UCLN     |
| 0.ANT  | 103 |      -2.3       |       -6.5        |    UCLN     |
| 0.ANT4 | 12  |      5.3*       |       5.9*        |    UCLN     |
|  0.PE  | 83  |      -82.1      |       12.4*       |    UCLN     |
| 0.PRE  |  8  |       TBD       |        TBD        |     TBD     |

Table:  Temporal signal detection and clock model selection with Bayesian Evaluation of Temporal Signal (BETS) {#tbl:bets_temporal_signal}

#### Rate Variation

Our approach of fitting nuanced models segregated by clade reveals that **the 'true' substitution rates of *Y. pestis* may be much higher than previously thought.** Previous work estimated that *Y. pestis* has one of the slowest observed substitution rates, around  1-2 x 10<sup>-8</sup>, which is on par with the exceptionally slow-evolving *Mycobacterium leprae* [@duchene2016GenomescaleRatesEvolutionary; @cui2013HistoricalVariationsMutation @spyrou2019PhylogeographySecondPlague]. The BETS analysis on the non-segregated data, which was highly unstable, fell within this published range with a 95% HPD between 1.16 x 10<sup>-8</sup> and 1.95 x 10<sup>-8</sup>. However, this global rate is a considerable underestimate, as clades with detectable temporal signal ranged from 2.33 x 10<sup>-8</sup> to 7.70 x 10<sup>-7</sup> (Table @tbl:bets_rate_cov_tmrca, Figure @fig:substitution_rate_boxplot).

> **Kat's Notes:**<br>
> - 0.PE appears to be an outlier, but wait for finalized logs.<br>
> - No differences with regards to rate/variation between pandemic and non-pandemic clades.<br>
>- I really want to see 1.IN, is there a progressive increase in rate along Branch 1?

It is interesting to note that several clades have distinctly different substitution rates (ie.non-overlapping estimates of the mean substitution rate in Figure @fig:substitution_rate_boxplot) and yet the relative amount of rate variation within a clade is similar (ie. overlapping coefficients of variation in Figure @fig:coefficient_of_variation_boxplot).

We hypothesize that outlier clades which are challenging to model (ex. 2.MED) have artificially decreased estimates of the mean substitution rate in past studies. This study therefore reports the substitution rate of *Y. pestis* to be much higher than previously thought and more comparable to bacteria such as *Mycobacterium tuberulcosis*.

|   Clade    |  N  |                 Substitution Rate                 | Coefficient of Variation |     tMRCA     |
|:----------:|:---:|:-------------------------------------------------:|:------------------------:|:-------------:|
|    All     | 191 |  1.16 x 10<sup>-8</sup> : 1.95 x 10<sup>-8</sup>  |       1.35 : 2.10        | -7400 : -3289 |
| 1.ORI  | 117 |  1.04 x 10<sup>-7</sup> : 1.53 x 10<sup>-7</sup>  |       1.02 :  1.87       |  1802 : 1907  |
|  1.IN  | 39  |                        --                         |            --            |      --       |
| 1.ANT  |  4  |  3.61 x 10<sup>-8</sup> : 9.63 x 10<sup>-8</sup>  |       0.00 : 2.12        |  1645 : 1889  |
| 1.PRE  | 40  |  3.69 x 10<sup>-8</sup> : 5.90 x 10<sup>-8</sup>  |       1.22 : 3.24        |  995 : 1324   |
| 2.MED  | 116 |  1.91 x 10<sup>-7</sup>, 3.01 x 10<sup>-7</sup>   |       2.23 : 3.50        |  1547 : 1867  |
| 2.ANT  | 54  |                        --                         |            --            |      --       |
| 4.ANT  | 11  |  3.59 x 10<sup>-8</sup> :  1.57 x 10<sup>-7</sup>  |       1.22 : 3.59        |  1847 : 1975  |
| 3.ANT  | 11  |                        --                         |            --            |      --       |
| 0.ANT  | 103 |                        --                         |            --            |      --       |
| 0.ANT4 | 12  |  2.33 x 10<sup>-8</sup> : 4.68 x 10<sup>-8</sup>  |       1.40 : 3.94        |   39 : 237    |
|  0.PE  | 83  | 4.32 x 10<sup>-7</sup> :   7.70 x 10<sup>-7</sup> |       1.70 : 2.78        |  1565 : 1892  |
| 0.PRE  |  8  | 3.95 x 10<sup>-8</sup> : 6.48  x 10<sup>-8</sup>  |       0.84 : 3.13        | -3070 : -2776 |

Table:  Estimate variation on the rate and tMRCA based on the 95% HPD. {#tbl:bets_rate_cov_tmrca}

![Mean substitution rate uncertainty by clade.](https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/4a67df7c718b/main/beast/all/chromosome/clade/logs_noHyperPrior/substitution_rate.png){#fig:substitution_rate_boxplot}

![Coefficient of variation uncertainty by clade.](https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/4a67df7c718b/main/beast/all/chromosome/clade/logs_noHyperPrior/coefficient_of_variation.png){#fig:coefficient_of_variation_boxplot}

#### Node Dating

To evaluate the dates associated with ancestral events, we annotated the maximum likelihood phylogeny with the estimated tMRCAs for clades with temporal signal (Figure @fig:divtree_node_dating). When re-contextualized into the global phylogeny, the node dates are topologically non-conflicting, meaning that parent nodes correctly pre-date child nodes. The sole exception is the estimated divergence of modern 0.PE which conflicts with the dates associated with the First and Second Pandemic clades. This conflict can be explained by several observations:

1. Clade 0.PE has the largest amount of uncertainty concerning the substitution rate.
1. Clade 0.PE has the greatest pairwise genetic distances, and longest root-to-tip distances to their MRCA.

>1. The single ancient sample of 0.PE was excluded from the BEAST analysis! This was partly by mistake, as for other clades, I separated out the ancient and modern samples based on their drastically different sampling periods. But for 0.PE, there's only one ancient sample and it got dropped.

![The maximum likelihood phylogeny annotated with the 95% HPD on clade root date.](https://raw.githubusercontent.com/ktmeaton/plague-phylogeography-projects/206f481/main/auspice/all/chromosome/full/filter5/divtree_node-dating.png){#fig:divtree_node_dating}

This drastic improvement in model performance reveals four intriguing aspects about the evolution of *Y. pestis*.

The first aspect is that **fitting a single clock model to the global phylogeny of *Y. pestis* is not statistically supported**. This can be observed in the relative instability of the MCMC analyses on the *reduced*  dataset, which fails to converge in parameter space. In contrast, successfully fitting models on a clade-by-clade basis reveals that **<i>Y. pestis</i> has more temporal signal than previously thought.** The observation that different populations have evolved at drastically different rates may explain the previous finding that the apparent structure in *Y. pestis* is dependent on dataset composition [@duchene2016GenomescaleRatesEvolutionary].


The final finding from constructing nuanced models concerns the outlier clades for which no detectable signal could be found, namely 0.ANT, 2.ANT, 3.ANT, 2.MED, and 1.IN. The reason for this lack of signal is unclear, but one explanation may be that **these *Y. pestis* populations are inappropriately separated based on the ```major branch``` and ```biovar``` systems**. We hypothesis that alternative strategies to subdivide these populations will yield greater insight, based on the methodological improvements demonstrated in this study.